{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "b'{\\n  \"name\" : \"phani-HP-Pavilion-x360-Convertible-14m-ba0xx\",\\n  \"cluster_name\" : \"elasticsearch\",\\n  \"cluster_uuid\" : \"73gQqRWET_WVj4GCsZxkYg\",\\n  \"version\" : {\\n    \"number\" : \"7.8.0\",\\n    \"build_flavor\" : \"default\",\\n    \"build_type\" : \"deb\",\\n    \"build_hash\" : \"757314695644ea9a1dc2fecd26d1a43856725e65\",\\n    \"build_date\" : \"2020-06-14T19:35:50.234439Z\",\\n    \"build_snapshot\" : false,\\n    \"lucene_version\" : \"8.5.1\",\\n    \"minimum_wire_compatibility_version\" : \"6.8.0\",\\n    \"minimum_index_compatibility_version\" : \"6.0.0-beta1\"\\n  },\\n  \"tagline\" : \"You Know, for Search\"\\n}\\n'\n"
     ]
    }
   ],
   "source": [
    "# make sure ES is up and running\n",
    "import requests\n",
    "res = requests.get('http://localhost:9200')\n",
    "print(res.content)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "#connect to our cluster\n",
    "from elasticsearch import Elasticsearch\n",
    "es = Elasticsearch([{'host': 'localhost', 'port': 9200}])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "# #let's iterate over swapi people documents and index them\n",
    "# import json\n",
    "# r = requests.get('http://localhost:9200') \n",
    "# i = 1\n",
    "# while r.status_code == 200:\n",
    "#     r = requests.get('http://swapi.co/api/people/'+ str(i))\n",
    "#     es.index(index='sw', doc_type='people', id=i, body=json.loads(r.content))\n",
    "#     i=i+1\n",
    " \n",
    "# print(i)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/phani/.local/lib/python3.8/site-packages/IPython/core/interactiveshell.py:3071: DtypeWarning: Columns (1,4,5,6,13,14,15,16) have mixed types.Specify dtype option on import or set low_memory=False.\n",
      "  has_raised = await self.run_ast_nodes(code_ast.body, cell_name,\n"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "df = pd.read_csv(\"/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Index(['cord_uid', 'sha', 'source_x', 'title', 'doi', 'pmcid', 'pubmed_id',\n",
       "       'license', 'abstract', 'publish_time', 'authors', 'journal', 'mag_id',\n",
       "       'who_covidence_id', 'arxiv_id', 'pdf_json_files', 'pmc_json_files',\n",
       "       'url', 's2_id'],\n",
       "      dtype='object')"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.columns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "rsp = es.search(index = \"metafile\",body = {\n",
    "            \"query\": { \n",
    "                \"bool\": { \n",
    "                    \"must\": [\n",
    "                        { \"match\": { 'abstract': 'diagnostics studies' }}\n",
    "                  ],\n",
    "                }\n",
    "              }\n",
    "            })"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'took': 1137,\n",
       " 'timed_out': False,\n",
       " '_shards': {'total': 1, 'successful': 1, 'skipped': 0, 'failed': 0},\n",
       " 'hits': {'total': {'value': 10000, 'relation': 'gte'},\n",
       "  'max_score': 10.399299,\n",
       "  'hits': [{'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'YTta93MBdq74P7QnDSo6',\n",
       "    '_score': 10.399299,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:59:00.330Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'cc-by-nc',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Rapid Molecular Tests for Influenza, Respiratory Syncytial Virus, and Other Respiratory Viruses: A Systematic Review of Diagnostic Accuracy and Clinical Impact Studies',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': 'ee341f85b6f787202910dc07f42cd5c08dada280',\n",
       "     'cord_uid': 'n95hz6jq',\n",
       "     'url': 'https://www.ncbi.nlm.nih.gov/pubmed/30689772/; https://doi.org/10.1093/cid/ciz056',\n",
       "     'source_x': 'Medline; PMC',\n",
       "     'pmc_json_files': 'document_parses/pmc_json/PMC7108200.xml.json',\n",
       "     'pdf_json_files': 'document_parses/pdf_json/ee341f85b6f787202910dc07f42cd5c08dada280.json',\n",
       "     'abstract': 'We systematically reviewed available evidence from Embase, Medline, and the Cochrane Library on diagnostic accuracy and clinical impact of commercially available rapid (results <3 hours) molecular diagnostics for respiratory viruses as compared to conventional molecular tests. Quality of included studies was assessed using the Quality Assessment of Diagnostic Accuracy Studies criteria for diagnostic test accuracy (DTA) studies, and the Cochrane Risk of Bias Assessment and Risk of Bias in Nonrandomized Studies of Interventions criteria for randomized and observational impact studies, respectively. Sixty-three DTA reports (56 studies) were meta-analyzed with a pooled sensitivity of 90.9% (95% confidence interval [CI], 88.7%–93.1%) and specificity of 96.1% (95% CI, 94.2%–97.9%) for the detection of either influenza virus (n = 29), respiratory syncytial virus (RSV) (n = 1), influenza virus and RSV (n = 19), or a viral panel including influenza virus and RSV (n = 14). The 15 included impact studies (5 randomized) were very heterogeneous and results were therefore inconclusive. However, we suggest that implementation of rapid diagnostics in hospital care settings should be considered.',\n",
       "     'journal': 'Clin Infect Dis',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC7108200',\n",
       "     'authors': 'Vos, Laura M; Bruning, Andrea H L; Reitsma, Johannes B; Schuurman, Rob; Riezebos-Brilman, Annelies; Hoepelman, Andy I M; Oosterheert, Jan Jelrik',\n",
       "     's2_id': '59339841',\n",
       "     'pubmed_id': '30689772',\n",
       "     'publish_time': '2019-10-01',\n",
       "     'doi': '10.1093/cid/ciz056',\n",
       "     'message': 'n95hz6jq,ee341f85b6f787202910dc07f42cd5c08dada280,Medline; PMC,\"Rapid Molecular Tests for Influenza, Respiratory Syncytial Virus, and Other Respiratory Viruses: A Systematic Review of Diagnostic Accuracy and Clinical Impact Studies\",10.1093/cid/ciz056,PMC7108200,30689772,cc-by-nc,\"We systematically reviewed available evidence from Embase, Medline, and the Cochrane Library on diagnostic accuracy and clinical impact of commercially available rapid (results <3 hours) molecular diagnostics for respiratory viruses as compared to conventional molecular tests. Quality of included studies was assessed using the Quality Assessment of Diagnostic Accuracy Studies criteria for diagnostic test accuracy (DTA) studies, and the Cochrane Risk of Bias Assessment and Risk of Bias in Nonrandomized Studies of Interventions criteria for randomized and observational impact studies, respectively. Sixty-three DTA reports (56 studies) were meta-analyzed with a pooled sensitivity of 90.9% (95% confidence interval [CI], 88.7%–93.1%) and specificity of 96.1% (95% CI, 94.2%–97.9%) for the detection of either influenza virus (n = 29), respiratory syncytial virus (RSV) (n = 1), influenza virus and RSV (n = 19), or a viral panel including influenza virus and RSV (n = 14). The 15 included impact studies (5 randomized) were very heterogeneous and results were therefore inconclusive. However, we suggest that implementation of rapid diagnostics in hospital care settings should be considered.\",2019-10-01,\"Vos, Laura M; Bruning, Andrea H L; Reitsma, Johannes B; Schuurman, Rob; Riezebos-Brilman, Annelies; Hoepelman, Andy I M; Oosterheert, Jan Jelrik\",Clin Infect Dis,,,,document_parses/pdf_json/ee341f85b6f787202910dc07f42cd5c08dada280.json,document_parses/pmc_json/PMC7108200.xml.json,https://www.ncbi.nlm.nih.gov/pubmed/30689772/; https://doi.org/10.1093/cid/ciz056,59339841\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'NTpY93MBdq74P7QneAbK',\n",
       "    '_score': 9.867579,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:57:16.616Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'unk',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Laboratory methods for determining pneumonia etiology in children.',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': None,\n",
       "     'cord_uid': 'n76k4nuh',\n",
       "     'url': 'https://doi.org/10.1093/cid/cir1073; https://www.ncbi.nlm.nih.gov/pubmed/22403229/',\n",
       "     'source_x': 'Medline',\n",
       "     'pmc_json_files': None,\n",
       "     'pdf_json_files': None,\n",
       "     'abstract': 'Laboratory diagnostics are a core component of any pneumonia etiology study. Recent advances in diagnostic technology have introduced newer methods that have greatly improved the ability to identify respiratory pathogens. However, determining the microbial etiology of pneumonia remains a challenge, especially in children. This is largely because of the inconsistent use of assays between studies, difficulties in specimen collection, and problems in interpreting the presence of pathogens as being causally related to the pneumonia event. The laboratory testing strategy for the Pneumonia Etiology Research for Child Health (PERCH) study aims to incorporate a broad range of diagnostic testing that will be standardized across the 7 participating sites. We describe the current status of laboratory diagnostics for pneumonia and the PERCH approach for specimen testing. Pneumonia diagnostics are evolving, and it is also a priority of PERCH to collect and archive specimens for future testing by promising diagnostic methods that are currently under development.',\n",
       "     'journal': 'Clinical infectious diseases : an official publication of the Infectious Diseases Society of America',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': None,\n",
       "     'authors': \"Murdoch, David R; O'Brien, Katherine L; Driscoll, Amanda J; Karron, Ruth A; Bhat, Niranjan\",\n",
       "     's2_id': '727177',\n",
       "     'pubmed_id': '22403229',\n",
       "     'publish_time': '2012',\n",
       "     'doi': '10.1093/cid/cir1073',\n",
       "     'message': 'n76k4nuh,,Medline,Laboratory methods for determining pneumonia etiology in children.,10.1093/cid/cir1073,,22403229,unk,\"Laboratory diagnostics are a core component of any pneumonia etiology study. Recent advances in diagnostic technology have introduced newer methods that have greatly improved the ability to identify respiratory pathogens. However, determining the microbial etiology of pneumonia remains a challenge, especially in children. This is largely because of the inconsistent use of assays between studies, difficulties in specimen collection, and problems in interpreting the presence of pathogens as being causally related to the pneumonia event. The laboratory testing strategy for the Pneumonia Etiology Research for Child Health (PERCH) study aims to incorporate a broad range of diagnostic testing that will be standardized across the 7 participating sites. We describe the current status of laboratory diagnostics for pneumonia and the PERCH approach for specimen testing. Pneumonia diagnostics are evolving, and it is also a priority of PERCH to collect and archive specimens for future testing by promising diagnostic methods that are currently under development.\",2012,\"Murdoch, David R; O\\'Brien, Katherine L; Driscoll, Amanda J; Karron, Ruth A; Bhat, Niranjan\",Clinical infectious diseases : an official publication of the Infectious Diseases Society of America,,,,,,https://doi.org/10.1093/cid/cir1073; https://www.ncbi.nlm.nih.gov/pubmed/22403229/,727177\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'cDhW93MBdq74P7QnRv-2',\n",
       "    '_score': 9.842388,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:54:37.684Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'bronze-oa',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Use and Evaluation of Molecular Diagnostics for Pneumonia Etiology Studies',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': 'd83f751c542a1f85ba6c4543824c2b261329521e',\n",
       "     'cord_uid': 'z5p3a64x',\n",
       "     'url': 'https://academic.oup.com/cid/article-pdf/54/suppl_2/S153/20938419/cir1060.pdf',\n",
       "     'source_x': 'PMC',\n",
       "     'pmc_json_files': 'document_parses/pmc_json/PMC3297547.xml.json',\n",
       "     'pdf_json_files': 'document_parses/pdf_json/d83f751c542a1f85ba6c4543824c2b261329521e.json',\n",
       "     'abstract': 'Comprehensive microbiological testing will be a core function of the Pneumonia Etiology Research for Child Health (PERCH) project. The development stage of PERCH provided the time and resources necessary for us to conduct a comprehensive review of the current state of respiratory diagnostics. These efforts allowed us to articulate the unique requirements of PERCH, establish that molecular methods would be central to our testing strategy, and focus on a short list of candidate platforms. This process also highlighted critical challenges in the general design and interpretation of diagnostic evaluation studies, particularly in the field of respiratory infections. Although our final molecular diagnostic platform was ultimately selected on the basis of operational and strategic considerations determined by the specific context of PERCH, our review highlighted several conceptual and practical challenges in respiratory diagnostics that have broader relevance for the performance and interpretation of pneumonia research studies.',\n",
       "     'journal': 'Clinical Infectious Diseases',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC3297547',\n",
       "     'authors': \"Bhat, Niranjan; O'Brien, Katherine L.; Karron, Ruth A.; Driscoll, Amanda J.; Murdoch, David R.\",\n",
       "     's2_id': None,\n",
       "     'pubmed_id': '22403230',\n",
       "     'publish_time': '2012-03-08',\n",
       "     'doi': '10.1093/cid/cir1060',\n",
       "     'message': 'z5p3a64x,d83f751c542a1f85ba6c4543824c2b261329521e,PMC,Use and Evaluation of Molecular Diagnostics for Pneumonia Etiology Studies,10.1093/cid/cir1060,PMC3297547,22403230,bronze-oa,\"Comprehensive microbiological testing will be a core function of the Pneumonia Etiology Research for Child Health (PERCH) project. The development stage of PERCH provided the time and resources necessary for us to conduct a comprehensive review of the current state of respiratory diagnostics. These efforts allowed us to articulate the unique requirements of PERCH, establish that molecular methods would be central to our testing strategy, and focus on a short list of candidate platforms. This process also highlighted critical challenges in the general design and interpretation of diagnostic evaluation studies, particularly in the field of respiratory infections. Although our final molecular diagnostic platform was ultimately selected on the basis of operational and strategic considerations determined by the specific context of PERCH, our review highlighted several conceptual and practical challenges in respiratory diagnostics that have broader relevance for the performance and interpretation of pneumonia research studies.\",2012-03-08,\"Bhat, Niranjan; O\\'Brien, Katherine L.; Karron, Ruth A.; Driscoll, Amanda J.; Murdoch, David R.\",Clinical Infectious Diseases,,,,document_parses/pdf_json/d83f751c542a1f85ba6c4543824c2b261329521e.json,document_parses/pmc_json/PMC3297547.xml.json,https://academic.oup.com/cid/article-pdf/54/suppl_2/S153/20938419/cir1060.pdf,\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'VTta93MBdq74P7Qn4NY8',\n",
       "    '_score': 9.8131695,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:59:54.365Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'no-cc',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Emerging Viral Infections',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': '20712c775ecc5b087dd6243208e5f47646e37756',\n",
       "     'cord_uid': 'm0xlnm6n',\n",
       "     'url': 'https://www.sciencedirect.com/science/article/pii/B9780123822277000975; https://api.elsevier.com/content/article/pii/B9780123822277000975',\n",
       "     'source_x': 'Elsevier; PMC',\n",
       "     'pmc_json_files': 'document_parses/pmc_json/PMC7150298.xml.json',\n",
       "     'pdf_json_files': 'document_parses/pdf_json/20712c775ecc5b087dd6243208e5f47646e37756.json',\n",
       "     'abstract': 'Emerging and re-emerging viral diseases pose a continuous health threat and disease burden to humans. Research efforts to mitigate the effects of infectious threats, focusing on improved surveillance and diagnostic capabilities, but also on the development of intervention strategies like the use of vaccines and antiviral agents, are crucial. We describe how genomic approaches may aid in mitigating viral threats, by highlighting potential roles and limitations of genomics approaches in virus surveillance and diagnostics, and development of vaccines and antivirals. Keywords antiviral, diagnostics, (emerging) infectious disease, evolution and diversity, gene expression profiling, genome-wide association studies, genome-wide functional screens, host genomics, prevention, surveillance, treatment, viral genomics, virus discovery, vaccine',\n",
       "     'journal': 'Genomic and Personalized Medicine',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC7150298',\n",
       "     'authors': 'Osterhaus, Albert D.M.E.; Smits, Saskia L.',\n",
       "     's2_id': '89396032',\n",
       "     'pubmed_id': None,\n",
       "     'publish_time': '2012-11-05',\n",
       "     'doi': '10.1016/b978-0-12-382227-7.00097-5',\n",
       "     'message': 'm0xlnm6n,20712c775ecc5b087dd6243208e5f47646e37756,Elsevier; PMC,Emerging Viral Infections,10.1016/b978-0-12-382227-7.00097-5,PMC7150298,,no-cc,\"Emerging and re-emerging viral diseases pose a continuous health threat and disease burden to humans. Research efforts to mitigate the effects of infectious threats, focusing on improved surveillance and diagnostic capabilities, but also on the development of intervention strategies like the use of vaccines and antiviral agents, are crucial. We describe how genomic approaches may aid in mitigating viral threats, by highlighting potential roles and limitations of genomics approaches in virus surveillance and diagnostics, and development of vaccines and antivirals. Keywords antiviral, diagnostics, (emerging) infectious disease, evolution and diversity, gene expression profiling, genome-wide association studies, genome-wide functional screens, host genomics, prevention, surveillance, treatment, viral genomics, virus discovery, vaccine\",2012-11-05,\"Osterhaus, Albert D.M.E.; Smits, Saskia L.\",Genomic and Personalized Medicine,,,,document_parses/pdf_json/20712c775ecc5b087dd6243208e5f47646e37756.json,document_parses/pmc_json/PMC7150298.xml.json,https://www.sciencedirect.com/science/article/pii/B9780123822277000975; https://api.elsevier.com/content/article/pii/B9780123822277000975,89396032\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'sjta93MBdq74P7QnwrhE',\n",
       "    '_score': 9.649883,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:59:46.682Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'no-cc',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Appropriate Use and Future Directions of Molecular Diagnostic Testing',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': '639e6993ca462dd015a5901c23c86faf8d5025da',\n",
       "     'cord_uid': 'ekok0mp5',\n",
       "     'url': 'https://doi.org/10.1007/s11908-020-0714-5; https://www.ncbi.nlm.nih.gov/pubmed/32030534/',\n",
       "     'source_x': 'Medline; PMC',\n",
       "     'pmc_json_files': 'document_parses/pmc_json/PMC7088562.xml.json',\n",
       "     'pdf_json_files': 'document_parses/pdf_json/639e6993ca462dd015a5901c23c86faf8d5025da.json',\n",
       "     'abstract': 'PURPOSE OF REVIEW: Major technologic advances in two main areas of molecular infectious disease diagnostics have resulted in accelerated adoption or ordering, outpacing implementation, and clinical utility studies. Physicians must understand the limitations to and appropriate utilization of these technologies in order to provide cost-effective and well-informed care for their patients. RECENT FINDINGS: Rapid molecular testing and, to a lesser degree, clinical metagenomics are now being routinely used in clinical practice. While these tests allow for a breadth of interrogation not possible with conventional microbiology, they pose new challenges for diagnostic and antimicrobial stewardship programs. This review will summarize the most recent literature on these two categories of technologic advances and discuss the few studies that have looked at utilization and stewardship approaches. This review also highlights the future directions for both of these technologies. SUMMARY: The appropriate utilization of rapid molecular testing and clinical metagenomics has not been well established. More studies are needed to assess their prospective impacts on patient management and antimicrobial stewardship efforts as the future state of infectious disease diagnostics will see continued expansion of these technologic advances.',\n",
       "     'journal': 'Curr Infect Dis Rep',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC7088562',\n",
       "     'authors': 'Graf, Erin H.; Pancholi, Preeti',\n",
       "     's2_id': '211045816',\n",
       "     'pubmed_id': '32030534',\n",
       "     'publish_time': '2020-02-06',\n",
       "     'doi': '10.1007/s11908-020-0714-5',\n",
       "     'message': 'ekok0mp5,639e6993ca462dd015a5901c23c86faf8d5025da,Medline; PMC,Appropriate Use and Future Directions of Molecular Diagnostic Testing,10.1007/s11908-020-0714-5,PMC7088562,32030534,no-cc,\"PURPOSE OF REVIEW: Major technologic advances in two main areas of molecular infectious disease diagnostics have resulted in accelerated adoption or ordering, outpacing implementation, and clinical utility studies. Physicians must understand the limitations to and appropriate utilization of these technologies in order to provide cost-effective and well-informed care for their patients. RECENT FINDINGS: Rapid molecular testing and, to a lesser degree, clinical metagenomics are now being routinely used in clinical practice. While these tests allow for a breadth of interrogation not possible with conventional microbiology, they pose new challenges for diagnostic and antimicrobial stewardship programs. This review will summarize the most recent literature on these two categories of technologic advances and discuss the few studies that have looked at utilization and stewardship approaches. This review also highlights the future directions for both of these technologies. SUMMARY: The appropriate utilization of rapid molecular testing and clinical metagenomics has not been well established. More studies are needed to assess their prospective impacts on patient management and antimicrobial stewardship efforts as the future state of infectious disease diagnostics will see continued expansion of these technologic advances.\",2020-02-06,\"Graf, Erin H.; Pancholi, Preeti\",Curr Infect Dis Rep,,,,document_parses/pdf_json/639e6993ca462dd015a5901c23c86faf8d5025da.json,document_parses/pmc_json/PMC7088562.xml.json,https://doi.org/10.1007/s11908-020-0714-5; https://www.ncbi.nlm.nih.gov/pubmed/32030534/,211045816\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'qDta93MBdq74P7Qnq6Ff',\n",
       "    '_score': 9.4985695,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:59:40.637Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'no-cc',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Clinical Perspectives on Targeting Therapies for Personalized Medicine',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': '936f8b5a192340e0cc33037e2da32e4541015475',\n",
       "     'cord_uid': '92xazjkb',\n",
       "     'url': 'https://api.elsevier.com/content/article/pii/S1876162315000917; https://www.ncbi.nlm.nih.gov/pubmed/26827603/; https://doi.org/10.1016/bs.apcsb.2015.11.003; https://www.sciencedirect.com/science/article/pii/S1876162315000917',\n",
       "     'source_x': 'Elsevier; Medline; PMC',\n",
       "     'pmc_json_files': 'document_parses/pmc_json/PMC7102676.xml.json',\n",
       "     'pdf_json_files': 'document_parses/pdf_json/936f8b5a192340e0cc33037e2da32e4541015475.json',\n",
       "     'abstract': 'Expected benefits from new technology include more efficient patient selection for clinical trials, more cost-effective treatment pathways for patients and health services and a more profitable accelerated approach for drug developers. Regulatory authorities expect the pharmaceutical and biotechnology industries to accelerate their development of companion diagnostics and companion therapeutics toward the goal of safer and more effective personalized medicine, and expect health services to fund and prescribers to adopt these new therapeutic technologies. This review discusses the importance of a range of new approaches to developing new and reprofiled medicines to treat common and serious diseases, and rare diseases: new network pharmacology approaches, adaptive trial designs with enriched populations more likely to respond safely to treatment, as assessed by companion diagnostics for response and toxicity risk and use of “real world” data. Case studies are described of single and multiple protein drug targets in several important therapeutic areas. These case studies also illustrate the value and complexity of use of selective biomarkers of clinical response and risk of adverse drug effects, either singly or in combination.',\n",
       "     'journal': 'Adv Protein Chem Struct Biol',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC7102676',\n",
       "     'authors': 'Singer, Donald R.J.; Zaïr, Zoulikha M.',\n",
       "     's2_id': '5962128',\n",
       "     'pubmed_id': '26827603',\n",
       "     'publish_time': '2015-12-29',\n",
       "     'doi': '10.1016/bs.apcsb.2015.11.003',\n",
       "     'message': '92xazjkb,936f8b5a192340e0cc33037e2da32e4541015475,Elsevier; Medline; PMC,Clinical Perspectives on Targeting Therapies for Personalized Medicine,10.1016/bs.apcsb.2015.11.003,PMC7102676,26827603,no-cc,\"Expected benefits from new technology include more efficient patient selection for clinical trials, more cost-effective treatment pathways for patients and health services and a more profitable accelerated approach for drug developers. Regulatory authorities expect the pharmaceutical and biotechnology industries to accelerate their development of companion diagnostics and companion therapeutics toward the goal of safer and more effective personalized medicine, and expect health services to fund and prescribers to adopt these new therapeutic technologies. This review discusses the importance of a range of new approaches to developing new and reprofiled medicines to treat common and serious diseases, and rare diseases: new network pharmacology approaches, adaptive trial designs with enriched populations more likely to respond safely to treatment, as assessed by companion diagnostics for response and toxicity risk and use of “real world” data. Case studies are described of single and multiple protein drug targets in several important therapeutic areas. These case studies also illustrate the value and complexity of use of selective biomarkers of clinical response and risk of adverse drug effects, either singly or in combination.\",2015-12-29,\"Singer, Donald R.J.; Zaïr, Zoulikha M.\",Adv Protein Chem Struct Biol,,,,document_parses/pdf_json/936f8b5a192340e0cc33037e2da32e4541015475.json,document_parses/pmc_json/PMC7102676.xml.json,https://api.elsevier.com/content/article/pii/S1876162315000917; https://www.ncbi.nlm.nih.gov/pubmed/26827603/; https://doi.org/10.1016/bs.apcsb.2015.11.003; https://www.sciencedirect.com/science/article/pii/S1876162315000917,5962128\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': '5TpZ93MBdq74P7QnI0Dg',\n",
       "    '_score': 9.233789,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:57:59.601Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'unk',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Recombinant protein-based assays for detection of antibodies to severe acute respiratory syndrome coronavirus spike and nucleocapsid proteins.',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': None,\n",
       "     'cord_uid': 'qfsxuyow',\n",
       "     'url': 'https://www.ncbi.nlm.nih.gov/pubmed/17229882/',\n",
       "     'source_x': 'Medline',\n",
       "     'pmc_json_files': None,\n",
       "     'pdf_json_files': None,\n",
       "     'abstract': 'Recombinant severe acute respiratory syndrome (SARS) nucleocapsid and spike protein-based immunoglobulin G immunoassays were developed and evaluated. Our assays demonstrated high sensitivity and specificity to the SARS coronavirus in sera collected from patients as late as 2 years postonset of symptoms. These assays will be useful not only for routine SARS coronavirus diagnostics but also for epidemiological and antibody kinetic studies.',\n",
       "     'journal': 'Clinical and vaccine immunology : CVI',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': None,\n",
       "     'authors': 'Haynes, Lia M; Miao, Congrong; Harcourt, Jennifer L; Montgomery, Joel M; Le, Mai Quynh; Dryga, Sergey A; Kamrud, Kurt I; Rivers, Bryan; Babcock, Gregory J; Oliver, Jennifer Betts; Comer, James A; Reynolds, Mary; Uyeki, Timothy M; Bausch, Daniel; Ksiazek, Thomas; Thomas, William; Alterson, Harold; Smith, Jonathan; Ambrosino, Donna M; Anderson, Larry J',\n",
       "     's2_id': '36317611',\n",
       "     'pubmed_id': '17229882',\n",
       "     'publish_time': '2007',\n",
       "     'doi': None,\n",
       "     'message': 'qfsxuyow,,Medline,Recombinant protein-based assays for detection of antibodies to severe acute respiratory syndrome coronavirus spike and nucleocapsid proteins.,,,17229882,unk,Recombinant severe acute respiratory syndrome (SARS) nucleocapsid and spike protein-based immunoglobulin G immunoassays were developed and evaluated. Our assays demonstrated high sensitivity and specificity to the SARS coronavirus in sera collected from patients as late as 2 years postonset of symptoms. These assays will be useful not only for routine SARS coronavirus diagnostics but also for epidemiological and antibody kinetic studies.,2007,\"Haynes, Lia M; Miao, Congrong; Harcourt, Jennifer L; Montgomery, Joel M; Le, Mai Quynh; Dryga, Sergey A; Kamrud, Kurt I; Rivers, Bryan; Babcock, Gregory J; Oliver, Jennifer Betts; Comer, James A; Reynolds, Mary; Uyeki, Timothy M; Bausch, Daniel; Ksiazek, Thomas; Thomas, William; Alterson, Harold; Smith, Jonathan; Ambrosino, Donna M; Anderson, Larry J\",Clinical and vaccine immunology : CVI,,,,,,https://www.ncbi.nlm.nih.gov/pubmed/17229882/,36317611\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'KDta93MBdq74P7QnIDxs',\n",
       "    '_score': 9.233789,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:59:04.774Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'no-cc',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Die Bedeutung von Coronaviren: Das Beispiel SARS',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': '2fcdc5c8c473ba298a0a62f5df84f8fd1c14a597',\n",
       "     'cord_uid': 'sqxmt699',\n",
       "     'url': 'https://www.ncbi.nlm.nih.gov/pubmed/15254819/',\n",
       "     'source_x': 'Medline; PMC',\n",
       "     'pmc_json_files': None,\n",
       "     'pdf_json_files': 'document_parses/pdf_json/2fcdc5c8c473ba298a0a62f5df84f8fd1c14a597.json',\n",
       "     'abstract': 'The emergence of the new SARS coronavirus has shown dramatically the high relevance of human coronavirus infections. This overview presents data of both epidemiological and clinical studies of this new virus infection. Moreover, details of research on the pathogenesis, diagnostics, and therapy are given. The recent infections in December 2003 clearly document that SARS will be a major topic also in 2004.',\n",
       "     'journal': 'Bundesgesundheitsblatt Gesundheitsforschung Gesundheitsschutz',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC7080169',\n",
       "     'authors': 'Schmitz, H.; Drosten, C.',\n",
       "     's2_id': '718363',\n",
       "     'pubmed_id': '15254819',\n",
       "     'publish_time': '2004',\n",
       "     'doi': '10.1007/s00103-004-0868-6',\n",
       "     'message': 'sqxmt699,2fcdc5c8c473ba298a0a62f5df84f8fd1c14a597,Medline; PMC,Die Bedeutung von Coronaviren: Das Beispiel SARS,10.1007/s00103-004-0868-6,PMC7080169,15254819,no-cc,\"The emergence of the new SARS coronavirus has shown dramatically the high relevance of human coronavirus infections. This overview presents data of both epidemiological and clinical studies of this new virus infection. Moreover, details of research on the pathogenesis, diagnostics, and therapy are given. The recent infections in December 2003 clearly document that SARS will be a major topic also in 2004.\",2004,\"Schmitz, H.; Drosten, C.\",Bundesgesundheitsblatt Gesundheitsforschung Gesundheitsschutz,,,,document_parses/pdf_json/2fcdc5c8c473ba298a0a62f5df84f8fd1c14a597.json,,https://www.ncbi.nlm.nih.gov/pubmed/15254819/,718363\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'Oztb93MBdq74P7QnEf1E',\n",
       "    '_score': 9.117379,\n",
       "    '_source': {'@timestamp': '2020-08-16T13:00:06.816Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'cc-by',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'The human viral challenge model: accelerating the evaluation of respiratory antivirals, vaccines and novel diagnostics',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': 'f13c88733ea45be9e923a282dfd42f8c277c187c',\n",
       "     'cord_uid': 'n026zwp8',\n",
       "     'url': 'https://www.ncbi.nlm.nih.gov/pubmed/29929556/; https://doi.org/10.1186/s12931-018-0784-1',\n",
       "     'source_x': 'Medline; PMC',\n",
       "     'pmc_json_files': 'document_parses/pmc_json/PMC6013893.xml.json',\n",
       "     'pdf_json_files': 'document_parses/pdf_json/f13c88733ea45be9e923a282dfd42f8c277c187c.json',\n",
       "     'abstract': 'The Human Viral Challenge (HVC) model has, for many decades, helped in the understanding of respiratory viruses and their role in disease pathogenesis. In a controlled setting using small numbers of volunteers removed from community exposure to other infections, this experimental model enables proof of concept work to be undertaken on novel therapeutics, including vaccines, immunomodulators and antivirals, as well as new diagnostics. Crucially, unlike conventional phase 1 studies, challenge studies include evaluable efficacy endpoints that then guide decisions on how to optimise subsequent field studies, as recommended by the FDA and thus licensing studies that follow. Such a strategy optimises the benefit of the studies and identifies possible threats early on, minimising the risk to subsequent volunteers but also maximising the benefit of scarce resources available to the research group investing in the research. Inspired by the principles of the 3Rs (Replacement, Reduction and Refinement) now commonly applied in the preclinical phase, HVC studies allow refinement and reduction of the subsequent development phase, accelerating progress towards further statistically powered phase 2b studies. The breadth of data generated from challenge studies allows for exploration of a wide range of variables and endpoints that can then be taken through to pivotal phase 3 studies. We describe the disease burden for acute respiratory viral infections for which current conventional development strategies have failed to produce therapeutics that meet clinical need. The Authors describe the HVC model’s utility in increasing scientific understanding and in progressing promising therapeutics through development. The contribution of the model to the elucidation of the virus-host interaction, both regarding viral pathogenicity and the body’s immunological response is discussed, along with its utility to assist in the development of novel diagnostics. Future applications of the model are also explored. ELECTRONIC SUPPLEMENTARY MATERIAL: The online version of this article (10.1186/s12931-018-0784-1) contains supplementary material, which is available to authorized users.',\n",
       "     'journal': 'Respir Res',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC6013893',\n",
       "     'authors': 'Lambkin-Williams, Rob; Noulin, Nicolas; Mann, Alex; Catchpole, Andrew; Gilbert, Anthony S.',\n",
       "     's2_id': '49356134',\n",
       "     'pubmed_id': '29929556',\n",
       "     'publish_time': '2018-06-22',\n",
       "     'doi': '10.1186/s12931-018-0784-1',\n",
       "     'message': 'n026zwp8,f13c88733ea45be9e923a282dfd42f8c277c187c,Medline; PMC,\"The human viral challenge model: accelerating the evaluation of respiratory antivirals, vaccines and novel diagnostics\",10.1186/s12931-018-0784-1,PMC6013893,29929556,cc-by,\"The Human Viral Challenge (HVC) model has, for many decades, helped in the understanding of respiratory viruses and their role in disease pathogenesis. In a controlled setting using small numbers of volunteers removed from community exposure to other infections, this experimental model enables proof of concept work to be undertaken on novel therapeutics, including vaccines, immunomodulators and antivirals, as well as new diagnostics. Crucially, unlike conventional phase 1 studies, challenge studies include evaluable efficacy endpoints that then guide decisions on how to optimise subsequent field studies, as recommended by the FDA and thus licensing studies that follow. Such a strategy optimises the benefit of the studies and identifies possible threats early on, minimising the risk to subsequent volunteers but also maximising the benefit of scarce resources available to the research group investing in the research. Inspired by the principles of the 3Rs (Replacement, Reduction and Refinement) now commonly applied in the preclinical phase, HVC studies allow refinement and reduction of the subsequent development phase, accelerating progress towards further statistically powered phase 2b studies. The breadth of data generated from challenge studies allows for exploration of a wide range of variables and endpoints that can then be taken through to pivotal phase 3 studies. We describe the disease burden for acute respiratory viral infections for which current conventional development strategies have failed to produce therapeutics that meet clinical need. The Authors describe the HVC model’s utility in increasing scientific understanding and in progressing promising therapeutics through development. The contribution of the model to the elucidation of the virus-host interaction, both regarding viral pathogenicity and the body’s immunological response is discussed, along with its utility to assist in the development of novel diagnostics. Future applications of the model are also explored. ELECTRONIC SUPPLEMENTARY MATERIAL: The online version of this article (10.1186/s12931-018-0784-1) contains supplementary material, which is available to authorized users.\",2018-06-22,\"Lambkin-Williams, Rob; Noulin, Nicolas; Mann, Alex; Catchpole, Andrew; Gilbert, Anthony S.\",Respir Res,,,,document_parses/pdf_json/f13c88733ea45be9e923a282dfd42f8c277c187c.json,document_parses/pmc_json/PMC6013893.xml.json,https://www.ncbi.nlm.nih.gov/pubmed/29929556/; https://doi.org/10.1186/s12931-018-0784-1,49356134\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'sTta93MBdq74P7QnIj13',\n",
       "    '_score': 9.024278,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:59:05.123Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'els-covid',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Advanced nanotechnologies in avian influenza: Current status and future trends – A review',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': 'a10ddac9f3991b1688ffa3278a2ec495db457d0e',\n",
       "     'cord_uid': 'mau5kyz5',\n",
       "     'url': 'https://www.ncbi.nlm.nih.gov/pubmed/28811028/; https://www.sciencedirect.com/science/article/pii/S0003267017307511; https://doi.org/10.1016/j.aca.2017.06.045; https://api.elsevier.com/content/article/pii/S0003267017307511',\n",
       "     'source_x': 'Elsevier; Medline; PMC',\n",
       "     'pmc_json_files': 'document_parses/pmc_json/PMC7094654.xml.json',\n",
       "     'pdf_json_files': 'document_parses/pdf_json/a10ddac9f3991b1688ffa3278a2ec495db457d0e.json',\n",
       "     'abstract': 'Abstract In the last decade, the control of avian influenza virus has experienced many difficulties, which have caused major global agricultural problems that have also led to public health consequences. Conventional biochemical methods are not sufficient to detect and control agricultural pathogens in the field due to the growing demand for food and subsidiary products; thus, studies aiming to develop potent alternatives to conventional biochemical methods are urgently needed. In this review, emerging detection systems, their applicability to diagnostics, and their therapeutic possibilities in view of nanotechnology are discussed. Nanotechnology-based sensors are used for rapid, sensitive and cost-effective diagnostics of agricultural pathogens. The application of different nanomaterials promotes interactions between these materials and the virus, which enables researchers to construct portable electroanalytical biosensing analyser that should effectively detect the influenza virus. The present review will provide insights into the guidelines for future experiments to develop better techniques to detect and control influenza viruses.',\n",
       "     'journal': 'Analytica Chimica Acta',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC7094654',\n",
       "     'authors': 'Moulick, Amitava; Richtera, Lukas; Milosavljevic, Vedran; Cernei, Natalia; Haddad, Yazan; Zitka, Ondrej; Kopel, Pavel; Heger, Zbynek; Adam, Vojtech',\n",
       "     's2_id': '42280538',\n",
       "     'pubmed_id': '28811028',\n",
       "     'publish_time': '2017-08-29',\n",
       "     'doi': '10.1016/j.aca.2017.06.045',\n",
       "     'message': 'mau5kyz5,a10ddac9f3991b1688ffa3278a2ec495db457d0e,Elsevier; Medline; PMC,Advanced nanotechnologies in avian influenza: Current status and future trends – A review,10.1016/j.aca.2017.06.045,PMC7094654,28811028,els-covid,\"Abstract In the last decade, the control of avian influenza virus has experienced many difficulties, which have caused major global agricultural problems that have also led to public health consequences. Conventional biochemical methods are not sufficient to detect and control agricultural pathogens in the field due to the growing demand for food and subsidiary products; thus, studies aiming to develop potent alternatives to conventional biochemical methods are urgently needed. In this review, emerging detection systems, their applicability to diagnostics, and their therapeutic possibilities in view of nanotechnology are discussed. Nanotechnology-based sensors are used for rapid, sensitive and cost-effective diagnostics of agricultural pathogens. The application of different nanomaterials promotes interactions between these materials and the virus, which enables researchers to construct portable electroanalytical biosensing analyser that should effectively detect the influenza virus. The present review will provide insights into the guidelines for future experiments to develop better techniques to detect and control influenza viruses.\",2017-08-29,\"Moulick, Amitava; Richtera, Lukas; Milosavljevic, Vedran; Cernei, Natalia; Haddad, Yazan; Zitka, Ondrej; Kopel, Pavel; Heger, Zbynek; Adam, Vojtech\",Analytica Chimica Acta,,,,document_parses/pdf_json/a10ddac9f3991b1688ffa3278a2ec495db457d0e.json,document_parses/pmc_json/PMC7094654.xml.json,https://www.ncbi.nlm.nih.gov/pubmed/28811028/; https://www.sciencedirect.com/science/article/pii/S0003267017307511; https://doi.org/10.1016/j.aca.2017.06.045; https://api.elsevier.com/content/article/pii/S0003267017307511,42280538\\r'}}]}}"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#Getting records of diagnostics studies\n",
    "rsp"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "rsp1 = es.search(index = \"metafile\",body = {\n",
    "            \"query\": { \n",
    "                \"bool\": { \n",
    "                    \"must\": [\n",
    "                        { \"match\": { 'title': 'diagnostics studies' }}\n",
    "                  ],\n",
    "                }\n",
    "              }\n",
    "            })"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'took': 602,\n",
       " 'timed_out': False,\n",
       " '_shards': {'total': 1, 'successful': 1, 'skipped': 0, 'failed': 0},\n",
       " 'hits': {'total': {'value': 1458, 'relation': 'eq'},\n",
       "  'max_score': 12.933924,\n",
       "  'hits': [{'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'cDhW93MBdq74P7QnRv-2',\n",
       "    '_score': 12.933924,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:54:37.684Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'bronze-oa',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Use and Evaluation of Molecular Diagnostics for Pneumonia Etiology Studies',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': 'd83f751c542a1f85ba6c4543824c2b261329521e',\n",
       "     'cord_uid': 'z5p3a64x',\n",
       "     'url': 'https://academic.oup.com/cid/article-pdf/54/suppl_2/S153/20938419/cir1060.pdf',\n",
       "     'source_x': 'PMC',\n",
       "     'pmc_json_files': 'document_parses/pmc_json/PMC3297547.xml.json',\n",
       "     'pdf_json_files': 'document_parses/pdf_json/d83f751c542a1f85ba6c4543824c2b261329521e.json',\n",
       "     'abstract': 'Comprehensive microbiological testing will be a core function of the Pneumonia Etiology Research for Child Health (PERCH) project. The development stage of PERCH provided the time and resources necessary for us to conduct a comprehensive review of the current state of respiratory diagnostics. These efforts allowed us to articulate the unique requirements of PERCH, establish that molecular methods would be central to our testing strategy, and focus on a short list of candidate platforms. This process also highlighted critical challenges in the general design and interpretation of diagnostic evaluation studies, particularly in the field of respiratory infections. Although our final molecular diagnostic platform was ultimately selected on the basis of operational and strategic considerations determined by the specific context of PERCH, our review highlighted several conceptual and practical challenges in respiratory diagnostics that have broader relevance for the performance and interpretation of pneumonia research studies.',\n",
       "     'journal': 'Clinical Infectious Diseases',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC3297547',\n",
       "     'authors': \"Bhat, Niranjan; O'Brien, Katherine L.; Karron, Ruth A.; Driscoll, Amanda J.; Murdoch, David R.\",\n",
       "     's2_id': None,\n",
       "     'pubmed_id': '22403230',\n",
       "     'publish_time': '2012-03-08',\n",
       "     'doi': '10.1093/cid/cir1060',\n",
       "     'message': 'z5p3a64x,d83f751c542a1f85ba6c4543824c2b261329521e,PMC,Use and Evaluation of Molecular Diagnostics for Pneumonia Etiology Studies,10.1093/cid/cir1060,PMC3297547,22403230,bronze-oa,\"Comprehensive microbiological testing will be a core function of the Pneumonia Etiology Research for Child Health (PERCH) project. The development stage of PERCH provided the time and resources necessary for us to conduct a comprehensive review of the current state of respiratory diagnostics. These efforts allowed us to articulate the unique requirements of PERCH, establish that molecular methods would be central to our testing strategy, and focus on a short list of candidate platforms. This process also highlighted critical challenges in the general design and interpretation of diagnostic evaluation studies, particularly in the field of respiratory infections. Although our final molecular diagnostic platform was ultimately selected on the basis of operational and strategic considerations determined by the specific context of PERCH, our review highlighted several conceptual and practical challenges in respiratory diagnostics that have broader relevance for the performance and interpretation of pneumonia research studies.\",2012-03-08,\"Bhat, Niranjan; O\\'Brien, Katherine L.; Karron, Ruth A.; Driscoll, Amanda J.; Murdoch, David R.\",Clinical Infectious Diseases,,,,document_parses/pdf_json/d83f751c542a1f85ba6c4543824c2b261329521e.json,document_parses/pmc_json/PMC3297547.xml.json,https://academic.oup.com/cid/article-pdf/54/suppl_2/S153/20938419/cir1060.pdf,\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': '-jlW93MBdq74P7Qnfwba',\n",
       "    '_score': 9.673,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:55:04.011Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'cc-by',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Astrovirus Diagnostics',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': '515da9e83a07430922c9fe2a979c0698799aa56c',\n",
       "     'cord_uid': 'gfqbdj0h',\n",
       "     'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5294979/',\n",
       "     'source_x': 'PMC',\n",
       "     'pmc_json_files': 'document_parses/pmc_json/PMC5294979.xml.json',\n",
       "     'pdf_json_files': 'document_parses/pdf_json/515da9e83a07430922c9fe2a979c0698799aa56c.json',\n",
       "     'abstract': 'Various methods exist to detect an astrovirus infection. Current methods include electron microscopy (EM), cell culture, immunoassays, polymerase chain reaction (PCR) and various other molecular approaches that can be applied in the context of diagnostic or in surveillance studies. With the advent of metagenomics, novel human astrovirus (HAstV) strains have been found in immunocompromised individuals in association with central nervous system (CNS) infections. This work reviews the past and current methods for astrovirus detection and their uses in both research laboratories and for medical diagnostic purposes.',\n",
       "     'journal': 'Viruses',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC5294979',\n",
       "     'authors': 'Pérot, Philippe; Lecuit, Marc; Eloit, Marc',\n",
       "     's2_id': None,\n",
       "     'pubmed_id': '28085120',\n",
       "     'publish_time': '2017-01-13',\n",
       "     'doi': '10.3390/v9010010',\n",
       "     'message': 'gfqbdj0h,515da9e83a07430922c9fe2a979c0698799aa56c,PMC,Astrovirus Diagnostics,10.3390/v9010010,PMC5294979,28085120,cc-by,\"Various methods exist to detect an astrovirus infection. Current methods include electron microscopy (EM), cell culture, immunoassays, polymerase chain reaction (PCR) and various other molecular approaches that can be applied in the context of diagnostic or in surveillance studies. With the advent of metagenomics, novel human astrovirus (HAstV) strains have been found in immunocompromised individuals in association with central nervous system (CNS) infections. This work reviews the past and current methods for astrovirus detection and their uses in both research laboratories and for medical diagnostic purposes.\",2017-01-13,\"Pérot, Philippe; Lecuit, Marc; Eloit, Marc\",Viruses,,,,document_parses/pdf_json/515da9e83a07430922c9fe2a979c0698799aa56c.json,document_parses/pmc_json/PMC5294979.xml.json,https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5294979/,\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': '5zlW93MBdq74P7Qn7zVp',\n",
       "    '_score': 9.673,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:55:35.654Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'no-cc',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Nanomolecular Diagnostics',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': '44f5bf79fa41bef5e13f5b117df9e618a36613a5',\n",
       "     'cord_uid': 'xiep0udu',\n",
       "     'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7121078/',\n",
       "     'source_x': 'PMC',\n",
       "     'pmc_json_files': 'document_parses/pmc_json/PMC7121078.xml.json',\n",
       "     'pdf_json_files': 'document_parses/pdf_json/44f5bf79fa41bef5e13f5b117df9e618a36613a5.json',\n",
       "     'abstract': 'Clinical application of molecular technologies to elucidate, diagnose and monitor human diseases is referred to as molecular diagnosis. It is a broader term than DNA (deoxyribonucleic acid) diagnostics and refers to the use of technologies that use DNA, RNA (ribonucleic acid), genes or proteins as bases for diagnostic tests. The scope of the subject is much wider and includes in vivo imaging and diagnosis at single molecule level. A more detailed description of molecular diagnostics is presented elsewhere (Jain 2017a).',\n",
       "     'journal': 'The Handbook of Nanomedicine',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC7121078',\n",
       "     'authors': 'Jain, Kewal K.',\n",
       "     's2_id': None,\n",
       "     'pubmed_id': None,\n",
       "     'publish_time': '2017-03-21',\n",
       "     'doi': '10.1007/978-1-4939-6966-1_4',\n",
       "     'message': 'xiep0udu,44f5bf79fa41bef5e13f5b117df9e618a36613a5,PMC,Nanomolecular Diagnostics,10.1007/978-1-4939-6966-1_4,PMC7121078,,no-cc,\"Clinical application of molecular technologies to elucidate, diagnose and monitor human diseases is referred to as molecular diagnosis. It is a broader term than DNA (deoxyribonucleic acid) diagnostics and refers to the use of technologies that use DNA, RNA (ribonucleic acid), genes or proteins as bases for diagnostic tests. The scope of the subject is much wider and includes in vivo imaging and diagnosis at single molecule level. A more detailed description of molecular diagnostics is presented elsewhere (Jain 2017a).\",2017-03-21,\"Jain, Kewal K.\",The Handbook of Nanomedicine,,,,document_parses/pdf_json/44f5bf79fa41bef5e13f5b117df9e618a36613a5.json,document_parses/pmc_json/PMC7121078.xml.json,https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7121078/,\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': '4zlW93MBdq74P7Qn6jG5',\n",
       "    '_score': 9.673,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:55:34.740Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'no-cc',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Molecular Diagnostics',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': '44604221fab887d21a3c321a882f18e343dab703',\n",
       "     'cord_uid': 'a9a82of4',\n",
       "     'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7115026/',\n",
       "     'source_x': 'PMC',\n",
       "     'pmc_json_files': 'document_parses/pmc_json/PMC7115026.xml.json',\n",
       "     'pdf_json_files': 'document_parses/pdf_json/44604221fab887d21a3c321a882f18e343dab703.json',\n",
       "     'abstract': 'Effective and early management of diseases requires record of the history, behavioral parameters, and travel information. These are helpful for the diagnosis, prevention, and control of the disease. There have been several advancements in the methods for diagnosing infectious diseases. The wide spectrum of tests such as biochemical evaluation, microbiological tools, immunological and molecular biology techniques, etc., is available. Each type of diagnostic technique is strong and reliable in its own sense but poses certain limitations. These limitations may be complemented by using a combination of tests. Older techniques such as microscopy and culturing of organisms from clinical specimens are error-free but are very labor intensive and extremely time consuming. There is a need to develop rapid and sensitive tests that can be used in both high- and low-resource settings. Molecular diagnostics such as Western blot, ELISA, PCR, DNA, and protein microarrays are revolutionizing the clinical practice of infectious diseases. Their effects are significant in acute-care settings where timely and accurate diagnostic tools are critical for patient treatment decisions and outcomes.',\n",
       "     'journal': 'Basic and Applied Aspects of Biotechnology',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC7115026',\n",
       "     'authors': 'Gupta, Varsha; Sengupta, Manjistha; Prakash, Jaya; Tripathy, Baishnab Charan',\n",
       "     's2_id': None,\n",
       "     'pubmed_id': None,\n",
       "     'publish_time': '2016-10-23',\n",
       "     'doi': '10.1007/978-981-10-0875-7_9',\n",
       "     'message': 'a9a82of4,44604221fab887d21a3c321a882f18e343dab703,PMC,Molecular Diagnostics,10.1007/978-981-10-0875-7_9,PMC7115026,,no-cc,\"Effective and early management of diseases requires record of the history, behavioral parameters, and travel information. These are helpful for the diagnosis, prevention, and control of the disease. There have been several advancements in the methods for diagnosing infectious diseases. The wide spectrum of tests such as biochemical evaluation, microbiological tools, immunological and molecular biology techniques, etc., is available. Each type of diagnostic technique is strong and reliable in its own sense but poses certain limitations. These limitations may be complemented by using a combination of tests. Older techniques such as microscopy and culturing of organisms from clinical specimens are error-free but are very labor intensive and extremely time consuming. There is a need to develop rapid and sensitive tests that can be used in both high- and low-resource settings. Molecular diagnostics such as Western blot, ELISA, PCR, DNA, and protein microarrays are revolutionizing the clinical practice of infectious diseases. Their effects are significant in acute-care settings where timely and accurate diagnostic tools are critical for patient treatment decisions and outcomes.\",2016-10-23,\"Gupta, Varsha; Sengupta, Manjistha; Prakash, Jaya; Tripathy, Baishnab Charan\",Basic and Applied Aspects of Biotechnology,,,,document_parses/pdf_json/44604221fab887d21a3c321a882f18e343dab703.json,document_parses/pmc_json/PMC7115026.xml.json,https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7115026/,\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': '5jlW93MBdq74P7Qn6jG5',\n",
       "    '_score': 9.673,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:55:34.740Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'no-cc',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Viral Diagnostics',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': '2295000abf3ce91c79d5c9dcad787acc5098c5a0',\n",
       "     'cord_uid': '3tpq5zar',\n",
       "     'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7115029/',\n",
       "     'source_x': 'PMC',\n",
       "     'pmc_json_files': 'document_parses/pmc_json/PMC7115029.xml.json',\n",
       "     'pdf_json_files': 'document_parses/pdf_json/2295000abf3ce91c79d5c9dcad787acc5098c5a0.json',\n",
       "     'abstract': 'This chapter discusses recent developments in diagnostics for cytomegalovirus (CMV), Epstein-Barr virus (EBV), BK virus (BKV), community respiratory viruses (CRVs), parvovirus, hepatitis viruses, HIV, and other viral agents of importance in solid organ and hematopoietic stem cell transplantation.',\n",
       "     'journal': 'Principles and Practice of Transplant Infectious Diseases',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC7115029',\n",
       "     'authors': 'Avery, Robin K.; Yen-Lieberman, Belinda',\n",
       "     's2_id': None,\n",
       "     'pubmed_id': None,\n",
       "     'publish_time': '2018-12-08',\n",
       "     'doi': '10.1007/978-1-4939-9034-4_49',\n",
       "     'message': '3tpq5zar,2295000abf3ce91c79d5c9dcad787acc5098c5a0,PMC,Viral Diagnostics,10.1007/978-1-4939-9034-4_49,PMC7115029,,no-cc,\"This chapter discusses recent developments in diagnostics for cytomegalovirus (CMV), Epstein-Barr virus (EBV), BK virus (BKV), community respiratory viruses (CRVs), parvovirus, hepatitis viruses, HIV, and other viral agents of importance in solid organ and hematopoietic stem cell transplantation.\",2018-12-08,\"Avery, Robin K.; Yen-Lieberman, Belinda\",Principles and Practice of Transplant Infectious Diseases,,,,document_parses/pdf_json/2295000abf3ce91c79d5c9dcad787acc5098c5a0.json,document_parses/pmc_json/PMC7115029.xml.json,https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7115029/,\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'uTlW93MBdq74P7Qn-DzO',\n",
       "    '_score': 9.673,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:55:38.443Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'no-cc',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Nanomolecular Diagnostics',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': None,\n",
       "     'cord_uid': 'p80tuijs',\n",
       "     'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7123333/',\n",
       "     'source_x': 'PMC',\n",
       "     'pmc_json_files': None,\n",
       "     'pdf_json_files': None,\n",
       "     'abstract': None,\n",
       "     'journal': 'The Handbook of Nanomedicine',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC7123333',\n",
       "     'authors': 'Jain, Kewal K.',\n",
       "     's2_id': None,\n",
       "     'pubmed_id': None,\n",
       "     'publish_time': '2008',\n",
       "     'doi': '10.1007/978-1-60327-319-0_3',\n",
       "     'message': 'p80tuijs,,PMC,Nanomolecular Diagnostics,10.1007/978-1-60327-319-0_3,PMC7123333,,no-cc,,2008,\"Jain, Kewal K.\",The Handbook of Nanomedicine,,,,,,https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7123333/,\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': '2DlW93MBdq74P7Qn-z13',\n",
       "    '_score': 9.673,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:55:38.815Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'no-cc',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Nanomolecular Diagnostics',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': None,\n",
       "     'cord_uid': '9hlx36e9',\n",
       "     'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7123705/',\n",
       "     'source_x': 'PMC',\n",
       "     'pmc_json_files': 'document_parses/pmc_json/PMC7123705.xml.json',\n",
       "     'pdf_json_files': None,\n",
       "     'abstract': 'Clinical application of molecular technologies to elucidate, diagnose, and monitor human diseases is referred to as molecular diagnosis. It is a broader term than DNA (deoxyribonucleic acid) diagnostics and refers to the use of technologies that use DNA, RNA (ribonucleic acid), genes, or proteins as bases for diagnostic tests. The scope of the subject is much wider and includes in vivo imaging and diagnosis at the single-molecule level. A more detailed description of molecular diagnostics is presented elsewhere (Jain 2012a).',\n",
       "     'journal': 'The Handbook of Nanomedicine',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC7123705',\n",
       "     'authors': 'Jain, Kewal K.',\n",
       "     's2_id': None,\n",
       "     'pubmed_id': None,\n",
       "     'publish_time': '2012-06-12',\n",
       "     'doi': '10.1007/978-1-61779-983-9_4',\n",
       "     'message': '9hlx36e9,,PMC,Nanomolecular Diagnostics,10.1007/978-1-61779-983-9_4,PMC7123705,,no-cc,\"Clinical application of molecular technologies to elucidate, diagnose, and monitor human diseases is referred to as molecular diagnosis. It is a broader term than DNA (deoxyribonucleic acid) diagnostics and refers to the use of technologies that use DNA, RNA (ribonucleic acid), genes, or proteins as bases for diagnostic tests. The scope of the subject is much wider and includes in vivo imaging and diagnosis at the single-molecule level. A more detailed description of molecular diagnostics is presented elsewhere (Jain 2012a).\",2012-06-12,\"Jain, Kewal K.\",The Handbook of Nanomedicine,,,,,document_parses/pmc_json/PMC7123705.xml.json,https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7123705/,\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': '7jta93MBdq74P7QnRFns',\n",
       "    '_score': 9.245245,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:59:14.589Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'no-cc',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Future Veterinary Diagnostics',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': '67d360a0c376d9ea6148f03c08d349aef0ecd811',\n",
       "     'cord_uid': 'o976nab9',\n",
       "     'url': 'https://api.elsevier.com/content/article/pii/S1557506310000558; https://doi.org/10.1053/j.jepm.2010.05.006; https://www.sciencedirect.com/science/article/pii/S1557506310000558; https://www.ncbi.nlm.nih.gov/pubmed/32288673/',\n",
       "     'source_x': 'Elsevier; Medline; PMC',\n",
       "     'pmc_json_files': 'document_parses/pmc_json/PMC7106287.xml.json',\n",
       "     'pdf_json_files': 'document_parses/pdf_json/67d360a0c376d9ea6148f03c08d349aef0ecd811.json',\n",
       "     'abstract': 'The development of rapid, accurate, and sensitive diagnostic methods for detecting pathogens is the basis for treating, controlling, and eradicating infectious diseases of veterinary importance. Scientific and technological advancements have revolutionized the field of veterinary diagnostics. Genome sequencing has allowed efficient, sensitive, and specific diagnostic assays to be developed based on the detection of nucleic acids. The integration of advances in biochemistry, proteomics, engineering, and medicine offers enormous potential for the rapid and accurate diagnosis of viral, microbial, genetic, and metabolic disease. In the future, polymerase chain reaction assays, microarray testing, genomic analysis, and metabolic profiling will be accomplished in a rapid, portable, sensitive, and cost-efficient manner.',\n",
       "     'journal': 'J Exot Pet Med',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC7106287',\n",
       "     'authors': 'Dahlhausen, Bob',\n",
       "     's2_id': '85097729',\n",
       "     'pubmed_id': '32288673',\n",
       "     'publish_time': '2010-09-19',\n",
       "     'doi': '10.1053/j.jepm.2010.05.006',\n",
       "     'message': 'o976nab9,67d360a0c376d9ea6148f03c08d349aef0ecd811,Elsevier; Medline; PMC,Future Veterinary Diagnostics,10.1053/j.jepm.2010.05.006,PMC7106287,32288673,no-cc,\"The development of rapid, accurate, and sensitive diagnostic methods for detecting pathogens is the basis for treating, controlling, and eradicating infectious diseases of veterinary importance. Scientific and technological advancements have revolutionized the field of veterinary diagnostics. Genome sequencing has allowed efficient, sensitive, and specific diagnostic assays to be developed based on the detection of nucleic acids. The integration of advances in biochemistry, proteomics, engineering, and medicine offers enormous potential for the rapid and accurate diagnosis of viral, microbial, genetic, and metabolic disease. In the future, polymerase chain reaction assays, microarray testing, genomic analysis, and metabolic profiling will be accomplished in a rapid, portable, sensitive, and cost-efficient manner.\",2010-09-19,\"Dahlhausen, Bob\",J Exot Pet Med,,,,document_parses/pdf_json/67d360a0c376d9ea6148f03c08d349aef0ecd811.json,document_parses/pmc_json/PMC7106287.xml.json,https://api.elsevier.com/content/article/pii/S1557506310000558; https://doi.org/10.1053/j.jepm.2010.05.006; https://www.sciencedirect.com/science/article/pii/S1557506310000558; https://www.ncbi.nlm.nih.gov/pubmed/32288673/,85097729\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'NDlW93MBdq74P7Qn6zII',\n",
       "    '_score': 8.853721,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:55:34.866Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'no-cc',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Diagnostics en pathologie infectieuse',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': None,\n",
       "     'cord_uid': 'mgsdm9zb',\n",
       "     'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7117811/',\n",
       "     'source_x': 'PMC',\n",
       "     'pmc_json_files': None,\n",
       "     'pdf_json_files': None,\n",
       "     'abstract': None,\n",
       "     'journal': 'Reanimation',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC7117811',\n",
       "     'authors': None,\n",
       "     's2_id': None,\n",
       "     'pubmed_id': None,\n",
       "     'publish_time': '2010-12-17',\n",
       "     'doi': '10.1007/s13546-010-0194-y',\n",
       "     'message': 'mgsdm9zb,,PMC,Diagnostics en pathologie infectieuse,10.1007/s13546-010-0194-y,PMC7117811,,no-cc,,2010-12-17,,Reanimation,,,,,,https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7117811/,\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': '3DlW93MBdq74P7Qn7zVp',\n",
       "    '_score': 8.853721,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:55:35.654Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'no-cc',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Pharmacogenomics and Molecular Diagnostics',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': 'faf9eb06811bb4ff480633421ab08e09961d63ae',\n",
       "     'cord_uid': '2temzspz',\n",
       "     'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7121064/',\n",
       "     'source_x': 'PMC',\n",
       "     'pmc_json_files': 'document_parses/pmc_json/PMC7121064.xml.json',\n",
       "     'pdf_json_files': 'document_parses/pdf_json/faf9eb06811bb4ff480633421ab08e09961d63ae.json',\n",
       "     'abstract': 'To develop new Polymerase Chain Reaction (PCR)-based assays for nucleic acid detection for infectious diseases. Development of new assays on demand for emerging infectious diseases. For example, no proper nucleic acid-based tests exist for detection of H1N1 influenza virus. My lab, being the parasitology research unit aims to fill this gap by developing this ability.',\n",
       "     'journal': 'Perspectives in Translational Research in Life Sciences and Biomedicine',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC7121064',\n",
       "     'authors': 'Banerjee, Ena Ray',\n",
       "     's2_id': None,\n",
       "     'pubmed_id': None,\n",
       "     'publish_time': '2017-10-18',\n",
       "     'doi': '10.1007/978-981-10-5870-7_5',\n",
       "     'message': '2temzspz,faf9eb06811bb4ff480633421ab08e09961d63ae,PMC,Pharmacogenomics and Molecular Diagnostics,10.1007/978-981-10-5870-7_5,PMC7121064,,no-cc,\"To develop new Polymerase Chain Reaction (PCR)-based assays for nucleic acid detection for infectious diseases. Development of new assays on demand for emerging infectious diseases. For example, no proper nucleic acid-based tests exist for detection of H1N1 influenza virus. My lab, being the parasitology research unit aims to fill this gap by developing this ability.\",2017-10-18,\"Banerjee, Ena Ray\",Perspectives in Translational Research in Life Sciences and Biomedicine,,,,document_parses/pdf_json/faf9eb06811bb4ff480633421ab08e09961d63ae.json,document_parses/pmc_json/PMC7121064.xml.json,https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7121064/,\\r'}}]}}"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "rsp1"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "pub_date_latest = es.search(index=\"metafile\", body={\"_source\": [\"abstract\"], \n",
    "                                     \"query\": {\"range\": \n",
    "                                               {\"publish_time\": \n",
    "                                                {\"gt\":\"2020-01-01\"}}}})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'took': 191,\n",
       " 'timed_out': False,\n",
       " '_shards': {'total': 1, 'successful': 1, 'skipped': 0, 'failed': 0},\n",
       " 'hits': {'total': {'value': 10000, 'relation': 'gte'},\n",
       "  'max_score': 1.0,\n",
       "  'hits': [{'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'ODlW93MBdq74P7Qnkwwa',\n",
       "    '_score': 1.0,\n",
       "    '_source': {'abstract': 'BACKGROUND: In pediatric cardiac anesthesiology, there is increased focus on minimizing morbidity, ensuring optimal functional status, and using health care resources sparingly. One aspect of care that has potential to affect all of the above is postoperative mechanical ventilation. Historically, postoperative ventilation was considered a must for maintaining patient stability. Ironically, it is recognized that mechanical ventilation may increase risk of adverse outcomes in the postoperative period. Hence, many institutions have advocated for immediate extubation or early extubation after many congenital heart surgeries which was first reported decades ago. METHODS: 637 consecutive patient charts were reviewed for pediatric patients undergoing cardiac surgery with cardiopulmonary bypass. Patients were placed into three groups. Those that were extubated in the operating room (OR) at the conclusion of surgery (Immediate Extubation or IE), those that were extubated within six hours of admission to the ICU (Early Extubation or EE) and those that were extubated sometime after six hours (Delayed Extubation or DE). Multiple variables were then recorded to see which factors correlated with successful Immediate or Early Extubation. RESULTS: Overall, 338 patients (53.1%) had IE), 273 (42.8%) had DE while only 26 patients (4.1%) had EE. The median age was 1174 days for the IE patients, 39 days for the DE patients, whereas 194 days for EE patients (p < 0.001). Weight and length were also significantly different in at least one extubation group from the other two (p < 0.001). The median ICU LOS was 3 and 4 days for IE and EE patients respectively, whereas it was 9.5 days for DE patients (p < 0.001). DE group had a significant longer median anesthesia time and cardiopulmonary bypass time than the other two extubation groups (p > 63,826.88 < 0.001). Regional low flow perfusion, deep hypothermia, deep hypothermic circulatory arrest, redo sternotomy, use of other sedatives, furosemide, epinephrine, vasopressin, open chest, cardiopulmonary support, pulmonary edema, syndrome, as well as difficult intubation were significantly associated with delayed extubation (IE, EE or DE). CONCLUSIONS: Immediate and early extubation was significantly associated with several factors, including patient age and size, duration of CPB, use of certain anesthetic drugs, and the amount of blood loss and blood replacement. IE can be successfully accomplished in a majority of pediatric patients undergoing surgery for congenital heart disease, including in a minority of infants.'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'OTlW93MBdq74P7Qnkwwa',\n",
       "    '_score': 1.0,\n",
       "    '_source': {'abstract': 'BACKGROUND: Vaccines are the most reliable alternative to elicit sterile immunity against malaria but their development has been hindered by polymorphisms and strain-specificity in previously studied antigens. New vaccine candidates are therefore urgently needed. Highly conserved Plasmodium falciparum reticulocyte-binding protein homologue-5 (PfRH5) has been identified as a potential candidate for anti-disease vaccine development. PfRH5 is essential for erythrocyte invasion by merozoites and crucial for parasite survival. However, there is paucity of data on the extent of genetic variations on PfRH5 in field isolates of Plasmodium falciparum. This study described genetic polymorphisms at the high affinity binding polypeptides (HABPs) 36718, 36727, 36728 of PfRH5 in Nigerian isolates of P. falciparum. This study tested the hypothesis that only specific conserved B and T cell epitopes on PfRH5 HABPs are crucial for vaccine development. METHODS: One hundred and ninety-five microscopically confirmed P. falciparum samples collected in a prospective cross-sectional study of three different populations in Lagos, Nigeria. Genetic diversity and haplotype construct of Pfrh5 gene were determined using bi-directional sequencing approach. Tajima’s D and the ratio of nonsynonymous vs synonymous mutations were utilized to estimate the extent of balancing and directional selection in the pfrh5 gene. RESULTS: Sequence analysis revealed three haplotypes of PfRH5 with negative Tajima’s D and dN/dS value of − 1.717 and 0.011 ± 0.020, respectively. A single nucleotide polymorphism, SNP (G → A) at position 608 was observed, which resulted in a change of the amino acid cysteine at position 203 to tyrosine. Haplotype and nucleotide diversities were 0.318 ± 0.016 and 0.0046 ± 0.0001 while inter-population genetic differentiation ranged from 0.007 to 0.037. Five polypeptide variants were identified, the most frequent being KTKYH with a frequency of 51.3%. One B-cell epitope, 151 major histocompatibility complex (MHC) class II T-cell epitopes, four intrinsically unstructured regions (IURs) and six MHC class I T-cell epitopes were observed in the study. Phylogenetic analysis of the sequences showed clustering and evidence of evolutionary relationship with 3D7, PAS-2 and FCB-2 RH5 sequences. CONCLUSIONS: This study has revealed low level of genetic polymorphisms in PfRH5 antigen with B- and T-cell epitopes in intrinsically unstructured regions along the PfRH5 gene in Lagos, Nigeria. A broader investigation is however required in other parts of the country to support the possible inclusion of PfRH5 in a cross-protective multi-component vaccine.'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'OjlW93MBdq74P7Qnkwwa',\n",
       "    '_score': 1.0,\n",
       "    '_source': {'abstract': 'The brainstem conveys sensory and motor inputs between the spinal cord and the brain, and contains nuclei of the cranial nerves. It controls the sleep-wake cycle and vital functions via the ascending reticular activating system and the autonomic nuclei, respectively. Brainstem dysfunction may lead to sensory and motor deficits, cranial nerve palsies, impairment of consciousness, dysautonomia, and respiratory failure. The brainstem is prone to various primary and secondary insults, resulting in acute or chronic dysfunction. Of particular importance for characterizing brainstem dysfunction and identifying the underlying etiology are a detailed clinical examination, MRI, neurophysiologic tests such as brainstem auditory evoked potentials, and an analysis of the cerebrospinal fluid. Detection of brainstem dysfunction is challenging but of utmost importance in comatose and deeply sedated patients both to guide therapy and to support outcome prediction. In the present review, we summarize the neuroanatomy, clinical syndromes, and diagnostic techniques of critical illness-associated brainstem dysfunction for the critical care setting.'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'OzlW93MBdq74P7Qnkwwa',\n",
       "    '_score': 1.0,\n",
       "    '_source': {'abstract': 'BACKGROUND: Growth hormone inducible transmembrane protein (GHITM) is a highly conserved transmembrane protein. This study was conducted to investigate the role of GHITM gene in the apoptosis and growth of the golden apple snail Pomacea canaliculate. RESULTS: The complete cDNA of this gene was cloned using the rapid amplification of cDNA ends (RACE) method and subjected to bioinformatics analysis. The full-length cDNA was 2242 bp, including an open reading frame of 1021 bp that encoded a protein of 342 amino acid residues. The mRNA expression profiles of GHITM gene in different tissues (liver, kidney, gonad and foot) and different growth phases (6-months old and 2-years old) showed that it was expressed in various tissues and different growth phases. Silencing of the GHITM gene by RNAi (RNA interference) experiments revealed that the GHITM gene possibly plays a role in inhibiting apoptosis through detecting the Caspase (Cysteine-requiring Aspartate Protease)-3 activity. In addition, the aperture width and body whorl length of the snail was significantly affected by RNAi, suggesting that this gene plays a significant role in promoting the growth of the organism. CONCLUSIONS: These results demonstrated that the GHITM gene was involved in apoptosis and growth in golden apple snail.'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'PDlW93MBdq74P7Qnkwwa',\n",
       "    '_score': 1.0,\n",
       "    '_source': {'abstract': 'OBJECTIVE: To examine the extent to which acute care hospitals in the Netherlands have adopted recommended practices to prevent catheter-associated urinary tract infection (CAUTI), central line-associated bloodstream infection (CLABSI), ventilator-associated pneumonia (VAP), and Clostridioides difficile infection (CDI). METHODS: Between 18 July 2017 and 31 October 2017, we surveyed the infection prevention teams of all acute care hospitals in the Netherlands. The survey instrument was based on the ‘Translating Healthcare-Associated Infection Prevention Research into Practice’ (TRIP) questionnaire and adapted to the Dutch context. Descriptive statistics were used to examine the reported regular use of CAUTI, CLABSI, VAP, and CDI prevention practices as well as the hospital characteristics. RESULTS: Out of 72 eligible hospitals, 47 (65.3%) responded. Surveillance systems for monitoring CAUTI, CLABSI, VAP, and CDI were present in 17.8, 95.4, 26.2, and 77.3% of hospitals, respectively. Antimicrobial stewardship programs have been established in 91.5% of participating hospitals. For CAUTI, the majority of hospitals regularly used aseptic technique during catheter insertion (95%) and portable bladder ultrasound scanners (86.1%). Intermittent catheterization and catheter stop-orders were regularly used by 65.8 and 62.2% of hospitals. For CLABSI, all hospitals regularly used maximum sterile barrier precautions and chlorhexidine gluconate for insertion site antisepsis. Avoidance of the femoral site for central line insertions was regularly used by 65.9% of hospitals. Urinary catheters and central-lines impregnated with antibiotics or antiseptics were rarely used (≤ 5%). Selective decontamination strategies for preventing VAP were used in 84% of hospitals. With the exception of disposable thermometers (31.8%), all prevention practices to prevent CDI were regularly used by more than 80% of hospitals. CONCLUSIONS: Most Dutch hospitals report regular use of recommended practices for preventing CLABSI and CDI. Several specific practices to prevent CAUTI and VAP were less frequently used, however, providing an opportunity for improvement.'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'QzlW93MBdq74P7Qnkwwa',\n",
       "    '_score': 1.0,\n",
       "    '_source': {'abstract': 'BACKGROUND: Randomised controlled trials (RCTs) provide the most reliable information to inform clinical practice and patient care. We aimed to map global clinical research publication activity through RCT-related articles in high-impact-factor medical journals over the past five decades. METHODS: We conducted a cross-sectional analysis of articles published in the highest ranked medical journals with an impact factor > 10 (according to Journal Citation Reports published in 2017). We searched PubMed/MEDLINE (from inception to December 31, 2017) for all RCT-related articles (e.g. primary RCTs, secondary analyses and methodology papers) published in high-impact-factor medical journals. For each included article, raw metadata were abstracted from the Web of Science. A process of standardization was conducted to unify the different terms and grammatical variants and to remove typographical, transcription and/or indexing errors. Descriptive analyses were conducted (including the number of articles, citations, most prolific authors, countries, journals, funding sources and keywords). Network analyses of collaborations between countries and co-words are presented. RESULTS: We included 39,305 articles (for the period 1965–2017) published in forty journals. The Lancet (n = 3593; 9.1%), the Journal of Clinical Oncology (n = 3343; 8.5%) and The New England Journal of Medicine (n = 3275 articles; 8.3%) published the largest number of RCTs. A total of 154 countries were involved in the production of articles. The global productivity ranking was led by the United States (n = 18,393 articles), followed by the United Kingdom (n = 8028 articles), Canada (n = 4548 articles) and Germany (n = 4415 articles). Seventeen authors who had published 100 or more articles were identified; the most prolific authors were affiliated with Duke University (United States), Harvard University (United States) and McMaster University (Canada). The main funding institutions were the National Institutes of Health (United States), Hoffmann-La Roche (Switzerland), Pfizer (United States), Merck Sharp & Dohme (United States) and Novartis (Switzerland). The 100 most cited RCTs were published in nine journals, led by The New England Journal of Medicine (n = 78 articles), The Lancet (n = 9 articles) and JAMA (n = 7 articles). These landmark contributions focused on novel methodological approaches (e.g. the “Bland-Altman method”) and trials on the management of chronic conditions (e.g. diabetes control, hormone replacement therapy in postmenopausal women, multiple therapies for diverse cancers, cardiovascular therapies such as lipid-lowering statins, antihypertensive medications, and antiplatelet and antithrombotic therapy). CONCLUSIONS: Our analysis identified authors, countries, funding institutions, landmark contributions and high-impact-factor medical journals publishing RCTs. Over the last 50 years, publication production in leading medical journals has increased, with Western countries leading in research but with low- and middle-income countries showing very limited representation.'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'RDlW93MBdq74P7Qnkwwa',\n",
       "    '_score': 1.0,\n",
       "    '_source': {'abstract': 'BACKGROUND: Professional caregivers working in child and youth welfare institutions are frequently faced with the complex mental health issues, emotional needs and challenging coping strategies of clients with cumulated traumatic experiences, leaving them prone to developing high levels of stress, burn-out and compassion fatigue. Trauma-informed care (TIC) is a milieu-therapeutic approach that aims to promote the self-efficacy and self-care of youth welfare staff by guiding them to a better understanding of their own and their clients’ stress symptoms and countertransference. Despite increasing efforts to implement TIC practices, and more widespread recognition of their value in youth welfare systems, there is a lack of studies evaluating the effectiveness of this approach. The aim of this study was to assess the effects of TIC practices in youth welfare institutions on both the physiological stress of staff members and clients’ physical aggression towards their caregivers. . METHODS: Data was obtained from a longitudinal study investigating the effectiveness of TIC in 14 residential youth welfare institutions. Our sample consisted of 47 youth welfare employees (66.0% female) aged from 23 to 60 years (M = 37.4, SD = 10.4 years). Hair cortisol concentration (HCC) and occurrences of client physical aggression were assessed at four annual measurement time points (T1 to T4). RESULTS: Participants in five institutions employing TIC practices (intervention group) showed significantly lower HCC at T4 than staff members from institutions who did not receive training in TIC (control group), indicating reduced physiological stress levels. At T4, the intervention group reported significantly less physical aggression than the control group. CONCLUSIONS: TIC might be a promising approach for reducing the emotional burden of employees and institutions should invest in training their staff in TIC practices. More research is necessary, to investigate the benefits and efficacy of TIC, both to youths and staff members, and to foster a better understanding of which specific factors may contribute to stress reduction.'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'RTlW93MBdq74P7Qnkwwa',\n",
       "    '_score': 1.0,\n",
       "    '_source': {'abstract': 'BACKGROUND: Catheter-related bloodstream infections (CR-BSI) cause high neonatal mortality and are related to inadequate aseptic technique during the care and maintenance of a catheter. The incidence of CR-BSI among neonates in Hung Vuong Hospital was higher than that of other neonatal care centres in Vietnam. METHODS: An 18-month pre- and post-intervention study was conducted over three 6-month periods to evaluate the effectiveness of the intervention for CR-BSI and to identify risk factors associated with CR-BSI. During the intervention period, we trained all nurses in the Department of Neonatology on BSI preventive practices, provided auditing and feedback about aseptic technique during catheter care and maintenance, and reorganised preparation of total parenteral nutrition. All neonates with intravenous catheter insertion ≥48 h in the pre- and post-intervention period were enrolled. A standardised questionnaire was used to collect data. Blood samples were collected for cultures. We used Poisson regression to calculate rate ratio (RR) and 95% confidence interval (CI) for CR-BSI incidence rates and logistic regression to identify risk factors associated with CR-BSI. RESULTS: Of 2225 neonates enrolled, 1027 were enrolled in the pre-intervention period, of which 53 CR-BSI cases occurred in 8399 catheter-days, and 1198 were enrolled in the post-intervention period, of which 32 CR-BSI cases occurred in 8324 catheter-days. Incidence rates of CR-BSI significantly decreased after the intervention (RR = 0.61, 95% CI 0.39–0.94). Days of hospitalisation, episodes of non-catheter–related hospital-acquired infections, and the proportion of deaths significantly decreased after the intervention (p < 0.01). The CR-BSI was associated with days of intravenous catheter (odds ratio [OR] = 1.05, 95% CI 1.03–1.08), use of endotracheal intubation (OR = 2.27, 95% CI 1.27–4.06), and intravenous injection (OR = 8.50, 95% CI 1.14–63.4). CONCLUSIONS: The interventions significantly decreased the incidence rate of CR-BSI. Regular refresher training and auditing and feedback about aseptic technique during care and maintenance of catheters are critical to reducing CR-BSI.'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': '8DlW93MBdq74P7Qnlw2u',\n",
       "    '_score': 1.0,\n",
       "    '_source': {'abstract': 'The PLOS Medicine editors discuss prospects for health and development in the coming decade.'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': '8jlW93MBdq74P7Qnlw2u',\n",
       "    '_score': 1.0,\n",
       "    '_source': {'abstract': 'Concerted evolution is often observed in multigene families such as the CEA gene family. As a result, sequence similarity of paralogous genes is significantly higher than expected from their evolutionary distance. Gene conversion, a “copy paste” DNA repair mechanism that transfers sequences from one gene to another and homologous recombination are drivers of concerted evolution. Nevertheless, some gene family members escape concerted evolution and acquire sufficient sequence differences that orthologous genes can be assigned in descendant species. Reasons why some gene family members can escape while others are captured by concerted evolution are poorly understood. By analyzing the entire CEA gene family in cattle (Bos taurus) we identified a member (CEACAM32) that was created by gene duplication and cooption of a unique transmembrane domain exon in the most recent ancestor of ruminants. CEACAM32 shows a unique, testis-specific expression pattern. Phylogenetic analysis indicated that CEACAM32 is not involved in concerted evolution of CEACAM1 paralogs in ruminants. However, analysis of gene conversion events revealed that CEACAM32 is subject to gene conversion but remarkably, these events are found in the leader exon and intron sequences but not in exons coding for the Ig-like domains. These findings suggest that natural selection hinders gene conversion affecting protein sequences of the mature protein and thereby support escape of CEACAM32 from concerted evolution.'}}]}}"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pub_date_latest"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'name': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       " 'cluster_name': 'elasticsearch',\n",
       " 'cluster_uuid': '73gQqRWET_WVj4GCsZxkYg',\n",
       " 'version': {'number': '7.8.0',\n",
       "  'build_flavor': 'default',\n",
       "  'build_type': 'deb',\n",
       "  'build_hash': '757314695644ea9a1dc2fecd26d1a43856725e65',\n",
       "  'build_date': '2020-06-14T19:35:50.234439Z',\n",
       "  'build_snapshot': False,\n",
       "  'lucene_version': '8.5.1',\n",
       "  'minimum_wire_compatibility_version': '6.8.0',\n",
       "  'minimum_index_compatibility_version': '6.0.0-beta1'},\n",
       " 'tagline': 'You Know, for Search'}"
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "es.info()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'count': 204823,\n",
       " '_shards': {'total': 1, 'successful': 1, 'skipped': 0, 'failed': 0}}"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "es.count(index='metafile')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [],
   "source": [
    "#  es.search(index='metafile', body={\"terms\":{'field':\"abstract\"}})\n",
    "# es.search('metafile', _source=['url'])\n",
    "# result = es.search(index='metafile', body={\"query\": {\"term\":{'field':\"abstract\"}}})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'took': 25,\n",
       " 'timed_out': False,\n",
       " '_shards': {'total': 1, 'successful': 1, 'skipped': 0, 'failed': 0},\n",
       " 'hits': {'total': {'value': 10000, 'relation': 'gte'},\n",
       "  'max_score': 1.0,\n",
       "  'hits': [{'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'SThW93MBdq74P7QnRv-0',\n",
       "    '_score': 1.0,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:54:37.670Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'no-cc',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Molecular Mimicry as a Mechanism of Autoimmune Disease',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': '189c25db19ffe602167dce79662f4f3a55e9f592',\n",
       "     'cord_uid': '735b7rpv',\n",
       "     'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3266166/',\n",
       "     'source_x': 'PMC',\n",
       "     'pmc_json_files': 'document_parses/pmc_json/PMC3266166.xml.json',\n",
       "     'pdf_json_files': 'document_parses/pdf_json/189c25db19ffe602167dce79662f4f3a55e9f592.json',\n",
       "     'abstract': 'A variety of mechanisms have been suggested as the means by which infections can initiate and/or exacerbate autoimmune diseases. One mechanism is molecular mimicry, where a foreign antigen shares sequence or structural similarities with self-antigens. Molecular mimicry has typically been characterized on an antibody or T cell level. However, structural relatedness between pathogen and self does not account for T cell activation in a number of autoimmune diseases. A proposed mechanism that could have been misinterpreted for molecular mimicry is the expression of dual T cell receptors (TCR) on a single T cell. These T cells have dual reactivity to both foreign and self-antigens leaving the host vulnerable to foreign insults capable of triggering an autoimmune response. In this review, we briefly discuss what is known about molecular mimicry followed by a discussion of the current understanding of dual TCRs. Finally, we discuss three mechanisms, including molecular mimicry, dual TCRs, and chimeric TCRs, by which dual reactivity of the T cell may play a role in autoimmune diseases.',\n",
       "     'journal': 'Clin Rev Allergy Immunol',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC3266166',\n",
       "     'authors': 'Cusick, Matthew F.; Libbey, Jane E.; Fujinami, Robert S.',\n",
       "     's2_id': None,\n",
       "     'pubmed_id': '22095454',\n",
       "     'publish_time': '2011-11-19',\n",
       "     'doi': '10.1007/s12016-011-8294-7',\n",
       "     'message': '735b7rpv,189c25db19ffe602167dce79662f4f3a55e9f592,PMC,Molecular Mimicry as a Mechanism of Autoimmune Disease,10.1007/s12016-011-8294-7,PMC3266166,22095454,no-cc,\"A variety of mechanisms have been suggested as the means by which infections can initiate and/or exacerbate autoimmune diseases. One mechanism is molecular mimicry, where a foreign antigen shares sequence or structural similarities with self-antigens. Molecular mimicry has typically been characterized on an antibody or T cell level. However, structural relatedness between pathogen and self does not account for T cell activation in a number of autoimmune diseases. A proposed mechanism that could have been misinterpreted for molecular mimicry is the expression of dual T cell receptors (TCR) on a single T cell. These T cells have dual reactivity to both foreign and self-antigens leaving the host vulnerable to foreign insults capable of triggering an autoimmune response. In this review, we briefly discuss what is known about molecular mimicry followed by a discussion of the current understanding of dual TCRs. Finally, we discuss three mechanisms, including molecular mimicry, dual TCRs, and chimeric TCRs, by which dual reactivity of the T cell may play a role in autoimmune diseases.\",2011-11-19,\"Cusick, Matthew F.; Libbey, Jane E.; Fujinami, Robert S.\",Clin Rev Allergy Immunol,,,,document_parses/pdf_json/189c25db19ffe602167dce79662f4f3a55e9f592.json,document_parses/pmc_json/PMC3266166.xml.json,https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3266166/,\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'SjhW93MBdq74P7QnRv-0',\n",
       "    '_score': 1.0,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:54:37.670Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'cc-by',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Low usage of government healthcare facilities for acute respiratory infections in guatemala: implications for influenza surveillance',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': 'ce88c461bd735cb67a9a7f6dc4d3ec1418d7dffb',\n",
       "     'cord_uid': 'qfv8ifvl',\n",
       "     'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3267779/',\n",
       "     'source_x': 'PMC',\n",
       "     'pmc_json_files': 'document_parses/pmc_json/PMC3267779.xml.json',\n",
       "     'pdf_json_files': 'document_parses/pdf_json/ce88c461bd735cb67a9a7f6dc4d3ec1418d7dffb.json',\n",
       "     'abstract': 'BACKGROUND: Sentinel surveillance for severe acute respiratory infections in hospitals and influenza-like illness in ambulatory clinics is recommended to assist in global pandemic influenza preparedness. Healthcare utilization patterns will affect the generalizability of data from sentinel sites and the potential to use them to estimate burden of disease. The objective of this study was to measure healthcare utilization patterns in Guatemala to inform the establishment of a sentinel surveillance system for influenza and other respiratory infections, and allow estimation of disease burden. METHODS: We used a stratified, two-stage cluster survey sample to select 1200 households from the Department of Santa Rosa. Trained interviewers screened household residents for self-reported pneumonia in the last year and influenza-like illness (ILI) in the last month and asked about healthcare utilization for each illness episode. RESULTS: We surveyed 1131 (94%) households and 5449 residents between October and December 2006 and identified 323 (6%) cases of pneumonia and 628 (13%) cases of ILI. Treatment for pneumonia outside the home was sought by 92% of the children <5 years old and 73% of the persons aged five years and older. For both children <5 years old (53%) and persons aged five years and older (31%) who reported pneumonia, private clinics were the most frequently reported source of care. For ILI, treatment was sought outside the home by 81% of children <5 years old and 65% of persons aged five years and older. Government ambulatory clinics were the most frequently sought source of care for ILI both for children <5 years old (41%) and persons aged five years and older (36%). CONCLUSIONS: Sentinel surveillance for influenza and other respiratory infections based in government health facilities in Guatemala will significantly underestimate the burden of disease. Adjustment for healthcare utilization practices will permit more accurate estimation of the incidence of influenza and other respiratory pathogens in the community.',\n",
       "     'journal': 'BMC Public Health',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC3267779',\n",
       "     'authors': 'Lindblade, Kim A; Johnson, April J; Arvelo, Wences; Zhang, Xingyou; Jordan, Hannah T; Reyes, Lissette; Fry, Alicia M; Padilla, Norma',\n",
       "     's2_id': None,\n",
       "     'pubmed_id': '22111590',\n",
       "     'publish_time': '2011-11-24',\n",
       "     'doi': '10.1186/1471-2458-11-885',\n",
       "     'message': 'qfv8ifvl,ce88c461bd735cb67a9a7f6dc4d3ec1418d7dffb,PMC,Low usage of government healthcare facilities for acute respiratory infections in guatemala: implications for influenza surveillance,10.1186/1471-2458-11-885,PMC3267779,22111590,cc-by,\"BACKGROUND: Sentinel surveillance for severe acute respiratory infections in hospitals and influenza-like illness in ambulatory clinics is recommended to assist in global pandemic influenza preparedness. Healthcare utilization patterns will affect the generalizability of data from sentinel sites and the potential to use them to estimate burden of disease. The objective of this study was to measure healthcare utilization patterns in Guatemala to inform the establishment of a sentinel surveillance system for influenza and other respiratory infections, and allow estimation of disease burden. METHODS: We used a stratified, two-stage cluster survey sample to select 1200 households from the Department of Santa Rosa. Trained interviewers screened household residents for self-reported pneumonia in the last year and influenza-like illness (ILI) in the last month and asked about healthcare utilization for each illness episode. RESULTS: We surveyed 1131 (94%) households and 5449 residents between October and December 2006 and identified 323 (6%) cases of pneumonia and 628 (13%) cases of ILI. Treatment for pneumonia outside the home was sought by 92% of the children <5 years old and 73% of the persons aged five years and older. For both children <5 years old (53%) and persons aged five years and older (31%) who reported pneumonia, private clinics were the most frequently reported source of care. For ILI, treatment was sought outside the home by 81% of children <5 years old and 65% of persons aged five years and older. Government ambulatory clinics were the most frequently sought source of care for ILI both for children <5 years old (41%) and persons aged five years and older (36%). CONCLUSIONS: Sentinel surveillance for influenza and other respiratory infections based in government health facilities in Guatemala will significantly underestimate the burden of disease. Adjustment for healthcare utilization practices will permit more accurate estimation of the incidence of influenza and other respiratory pathogens in the community.\",2011-11-24,\"Lindblade, Kim A; Johnson, April J; Arvelo, Wences; Zhang, Xingyou; Jordan, Hannah T; Reyes, Lissette; Fry, Alicia M; Padilla, Norma\",BMC Public Health,,,,document_parses/pdf_json/ce88c461bd735cb67a9a7f6dc4d3ec1418d7dffb.json,document_parses/pmc_json/PMC3267779.xml.json,https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3267779/,\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'SzhW93MBdq74P7QnRv-0',\n",
       "    '_score': 1.0,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:54:37.670Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'cc-by',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Understanding the clinical spectrum of complicated Plasmodium vivax malaria: a systematic review on the contributions of the Brazilian literature',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': '16924e33ce20f1a51ad925c2f1bf6ad6dce8cbbc',\n",
       "     'cord_uid': '271vldil',\n",
       "     'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3268102/',\n",
       "     'source_x': 'PMC',\n",
       "     'pmc_json_files': 'document_parses/pmc_json/PMC3268102.xml.json',\n",
       "     'pdf_json_files': 'document_parses/pdf_json/16924e33ce20f1a51ad925c2f1bf6ad6dce8cbbc.json',\n",
       "     'abstract': \"The resurgence of the malaria eradication agenda and the increasing number of severe manifestation reports has contributed to a renewed interested in the Plasmodium vivax infection. It is the most geographically widespread parasite causing human malaria, with around 2.85 billion people living under risk of infection. The Brazilian Amazon region reports more than 50% of the malaria cases in Latin America and since 1990 there is a marked predominance of this species, responsible for 85% of cases in 2009. However, only a few complicated cases of P. vivax have been reported from this region. A systematic review of the Brazilian indexed and non-indexed literature on complicated cases of vivax malaria was performed including published articles, masters' dissertations, doctoral theses and national congresses' abstracts. The following information was retrieved: patient characteristics (demographic, presence of co-morbidities and, whenever possible, associated genetic disorders); description of each major clinical manifestation. As a result, 27 articles, 28 abstracts from scientific events' annals and 13 theses/dissertations were found, only after 1987. Most of the reported information was described in small case series and case reports of patients from all the Amazonian states, and also in travellers from Brazilian non-endemic areas. The more relevant clinical complications were anaemia, thrombocytopaenia, jaundice and acute respiratory distress syndrome, present in all age groups, in addition to other more rare clinical pictures. Complications in pregnant women were also reported. Acute and chronic co-morbidities were frequent, however death was occasional. Clinical atypical cases of malaria are more frequent than published in the indexed literature, probably due to a publication bias. In the Brazilian Amazon (considered to be a low to moderate intensity area of transmission), clinical data are in accordance with the recent findings of severity described in diverse P. vivax endemic areas (especially anaemia in Southeast Asia), however in this region both children and adults are affected. Finally, gaps of knowledge and areas for future research are opportunely pointed out.\",\n",
       "     'journal': 'Malar J',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC3268102',\n",
       "     'authors': 'Lacerda, Marcus VG; Mourão, Maria PG; Alexandre, Márcia AA; Siqueira, André M; Magalhães, Belisa ML; Martinez-Espinosa, Flor E; Santana Filho, Franklin S; Brasil, Patrícia; Ventura, Ana MRS; Tada, Mauro S; Couto, Vanja SCD; Silva, Antônio R; Silva, Rita SU; Alecrim, Maria GC',\n",
       "     's2_id': None,\n",
       "     'pubmed_id': '22230294',\n",
       "     'publish_time': '2012-01-09',\n",
       "     'doi': '10.1186/1475-2875-11-12',\n",
       "     'message': '271vldil,16924e33ce20f1a51ad925c2f1bf6ad6dce8cbbc,PMC,Understanding the clinical spectrum of complicated Plasmodium vivax malaria: a systematic review on the contributions of the Brazilian literature,10.1186/1475-2875-11-12,PMC3268102,22230294,cc-by,\"The resurgence of the malaria eradication agenda and the increasing number of severe manifestation reports has contributed to a renewed interested in the Plasmodium vivax infection. It is the most geographically widespread parasite causing human malaria, with around 2.85 billion people living under risk of infection. The Brazilian Amazon region reports more than 50% of the malaria cases in Latin America and since 1990 there is a marked predominance of this species, responsible for 85% of cases in 2009. However, only a few complicated cases of P. vivax have been reported from this region. A systematic review of the Brazilian indexed and non-indexed literature on complicated cases of vivax malaria was performed including published articles, masters\\' dissertations, doctoral theses and national congresses\\' abstracts. The following information was retrieved: patient characteristics (demographic, presence of co-morbidities and, whenever possible, associated genetic disorders); description of each major clinical manifestation. As a result, 27 articles, 28 abstracts from scientific events\\' annals and 13 theses/dissertations were found, only after 1987. Most of the reported information was described in small case series and case reports of patients from all the Amazonian states, and also in travellers from Brazilian non-endemic areas. The more relevant clinical complications were anaemia, thrombocytopaenia, jaundice and acute respiratory distress syndrome, present in all age groups, in addition to other more rare clinical pictures. Complications in pregnant women were also reported. Acute and chronic co-morbidities were frequent, however death was occasional. Clinical atypical cases of malaria are more frequent than published in the indexed literature, probably due to a publication bias. In the Brazilian Amazon (considered to be a low to moderate intensity area of transmission), clinical data are in accordance with the recent findings of severity described in diverse P. vivax endemic areas (especially anaemia in Southeast Asia), however in this region both children and adults are affected. Finally, gaps of knowledge and areas for future research are opportunely pointed out.\",2012-01-09,\"Lacerda, Marcus VG; Mourão, Maria PG; Alexandre, Márcia AA; Siqueira, André M; Magalhães, Belisa ML; Martinez-Espinosa, Flor E; Santana Filho, Franklin S; Brasil, Patrícia; Ventura, Ana MRS; Tada, Mauro S; Couto, Vanja SCD; Silva, Antônio R; Silva, Rita SU; Alecrim, Maria GC\",Malar J,,,,document_parses/pdf_json/16924e33ce20f1a51ad925c2f1bf6ad6dce8cbbc.json,document_parses/pmc_json/PMC3268102.xml.json,https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3268102/,\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'TDhW93MBdq74P7QnRv-0',\n",
       "    '_score': 1.0,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:54:37.672Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'cc-by',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Protective Role of the ACE2/Ang-(1–9) Axis in Cardiovascular Remodeling',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': '989be59a0cd5354610ff80be6e50fa6617c077bf',\n",
       "     'cord_uid': 'qk37euo4',\n",
       "     'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3270559/',\n",
       "     'source_x': 'PMC',\n",
       "     'pmc_json_files': 'document_parses/pmc_json/PMC3270559.xml.json',\n",
       "     'pdf_json_files': 'document_parses/pdf_json/989be59a0cd5354610ff80be6e50fa6617c077bf.json',\n",
       "     'abstract': 'Despite reduction in cardiovascular (CV) events and end-organ damage with the current pharmacologic strategies, CV disease remains the primary cause of death in the world. Pharmacological therapies based on the renin angiotensin system (RAS) blockade are used extensively for the treatment of hypertension, heart failure, and CV remodeling but in spite of their success the prevalence of end-organ damage and residual risk remain still high. Novel approaches must be discovered for a more effective treatment of residual CV remodeling and risk. The ACE2/Ang-(1–9) axis is a new and important target to counterbalance the vasoconstrictive/proliferative RAS axis. Ang-(1–9) is hydrolyzed slower than Ang-(1–7) and is able to bind the Ang II type 2 receptor. We review here the current experimental evidence suggesting that activation of the ACE2/Ang-(1–9) axis protects the heart and vessels (and possibly the kidney) from adverse cardiovascular remodeling in hypertension as well as in heart failure.',\n",
       "     'journal': 'Int J Hypertens',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC3270559',\n",
       "     'authors': 'Ocaranza, María Paz; Jalil, Jorge E.',\n",
       "     's2_id': None,\n",
       "     'pubmed_id': '22315665',\n",
       "     'publish_time': '2012-01-19',\n",
       "     'doi': '10.1155/2012/594361',\n",
       "     'message': 'qk37euo4,989be59a0cd5354610ff80be6e50fa6617c077bf,PMC,Protective Role of the ACE2/Ang-(1–9) Axis in Cardiovascular Remodeling,10.1155/2012/594361,PMC3270559,22315665,cc-by,\"Despite reduction in cardiovascular (CV) events and end-organ damage with the current pharmacologic strategies, CV disease remains the primary cause of death in the world. Pharmacological therapies based on the renin angiotensin system (RAS) blockade are used extensively for the treatment of hypertension, heart failure, and CV remodeling but in spite of their success the prevalence of end-organ damage and residual risk remain still high. Novel approaches must be discovered for a more effective treatment of residual CV remodeling and risk. The ACE2/Ang-(1–9) axis is a new and important target to counterbalance the vasoconstrictive/proliferative RAS axis. Ang-(1–9) is hydrolyzed slower than Ang-(1–7) and is able to bind the Ang II type 2 receptor. We review here the current experimental evidence suggesting that activation of the ACE2/Ang-(1–9) axis protects the heart and vessels (and possibly the kidney) from adverse cardiovascular remodeling in hypertension as well as in heart failure.\",2012-01-19,\"Ocaranza, María Paz; Jalil, Jorge E.\",Int J Hypertens,,,,document_parses/pdf_json/989be59a0cd5354610ff80be6e50fa6617c077bf.json,document_parses/pmc_json/PMC3270559.xml.json,https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3270559/,\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'TThW93MBdq74P7QnRv-0',\n",
       "    '_score': 1.0,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:54:37.673Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'cc-by',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Viral Proteins Acquired from a Host Converge to Simplified Domain Architectures',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': 'df0fec846e0a329190f5f8babcb5ecf535141338',\n",
       "     'cord_uid': 'gnkzb7qr',\n",
       "     'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3271019/',\n",
       "     'source_x': 'PMC',\n",
       "     'pmc_json_files': 'document_parses/pmc_json/PMC3271019.xml.json',\n",
       "     'pdf_json_files': 'document_parses/pdf_json/df0fec846e0a329190f5f8babcb5ecf535141338.json',\n",
       "     'abstract': 'The infection cycle of viruses creates many opportunities for the exchange of genetic material with the host. Many viruses integrate their sequences into the genome of their host for replication. These processes may lead to the virus acquisition of host sequences. Such sequences are prone to accumulation of mutations and deletions. However, in rare instances, sequences acquired from a host become beneficial for the virus. We searched for unexpected sequence similarity among the 900,000 viral proteins and all proteins from cellular organisms. Here, we focus on viruses that infect metazoa. The high-conservation analysis yielded 187 instances of highly similar viral-host sequences. Only a small number of them represent viruses that hijacked host sequences. The low-conservation sequence analysis utilizes the Pfam family collection. About 5% of the 12,000 statistical models archived in Pfam are composed of viral-metazoan proteins. In about half of Pfam families, we provide indirect support for the directionality from the host to the virus. The other families are either wrongly annotated or reflect an extensive sequence exchange between the viruses and their hosts. In about 75% of cross-taxa Pfam families, the viral proteins are significantly shorter than their metazoan counterparts. The tendency for shorter viral proteins relative to their related host proteins accounts for the acquisition of only a fragment of the host gene, the elimination of an internal domain and shortening of the linkers between domains. We conclude that, along viral evolution, the host-originated sequences accommodate simplified domain compositions. We postulate that the trimmed proteins act by interfering with the fundamental function of the host including intracellular signaling, post-translational modification, protein-protein interaction networks and cellular trafficking. We compiled a collection of hijacked protein sequences. These sequences are attractive targets for manipulation of viral infection.',\n",
       "     'journal': 'PLoS Comput Biol',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC3271019',\n",
       "     'authors': 'Rappoport, Nadav; Linial, Michal',\n",
       "     's2_id': None,\n",
       "     'pubmed_id': '22319434',\n",
       "     'publish_time': '2012-02-02',\n",
       "     'doi': '10.1371/journal.pcbi.1002364',\n",
       "     'message': 'gnkzb7qr,df0fec846e0a329190f5f8babcb5ecf535141338,PMC,Viral Proteins Acquired from a Host Converge to Simplified Domain Architectures,10.1371/journal.pcbi.1002364,PMC3271019,22319434,cc-by,\"The infection cycle of viruses creates many opportunities for the exchange of genetic material with the host. Many viruses integrate their sequences into the genome of their host for replication. These processes may lead to the virus acquisition of host sequences. Such sequences are prone to accumulation of mutations and deletions. However, in rare instances, sequences acquired from a host become beneficial for the virus. We searched for unexpected sequence similarity among the 900,000 viral proteins and all proteins from cellular organisms. Here, we focus on viruses that infect metazoa. The high-conservation analysis yielded 187 instances of highly similar viral-host sequences. Only a small number of them represent viruses that hijacked host sequences. The low-conservation sequence analysis utilizes the Pfam family collection. About 5% of the 12,000 statistical models archived in Pfam are composed of viral-metazoan proteins. In about half of Pfam families, we provide indirect support for the directionality from the host to the virus. The other families are either wrongly annotated or reflect an extensive sequence exchange between the viruses and their hosts. In about 75% of cross-taxa Pfam families, the viral proteins are significantly shorter than their metazoan counterparts. The tendency for shorter viral proteins relative to their related host proteins accounts for the acquisition of only a fragment of the host gene, the elimination of an internal domain and shortening of the linkers between domains. We conclude that, along viral evolution, the host-originated sequences accommodate simplified domain compositions. We postulate that the trimmed proteins act by interfering with the fundamental function of the host including intracellular signaling, post-translational modification, protein-protein interaction networks and cellular trafficking. We compiled a collection of hijacked protein sequences. These sequences are attractive targets for manipulation of viral infection.\",2012-02-02,\"Rappoport, Nadav; Linial, Michal\",PLoS Comput Biol,,,,document_parses/pdf_json/df0fec846e0a329190f5f8babcb5ecf535141338.json,document_parses/pmc_json/PMC3271019.xml.json,https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3271019/,\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'TjhW93MBdq74P7QnRv-0',\n",
       "    '_score': 1.0,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:54:37.673Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'no-cc',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Treatment of Neuroterrorism',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': '2a753c9ec1b4376606a18b951f4257a602248ddd',\n",
       "     'cord_uid': 'sgcf9v3m',\n",
       "     'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3271146/',\n",
       "     'source_x': 'PMC',\n",
       "     'pmc_json_files': 'document_parses/pmc_json/PMC3271146.xml.json',\n",
       "     'pdf_json_files': 'document_parses/pdf_json/2a753c9ec1b4376606a18b951f4257a602248ddd.json',\n",
       "     'abstract': 'Bioterrorism is defined as the intentional use of biological, chemical, nuclear, or radiological agents to cause disease, death, or environmental damage. Early recognition of a bioterrorist attack is of utmost importance to minimize casualties and initiate appropriate therapy. The range of agents that could potentially be used as weapons is wide, however, only a few of these agents have all the characteristics making them ideal for that purpose. Many of the chemical and biological weapons can cause neurological symptoms and damage the nervous system in varying degrees. Therefore, preparedness among neurologists is important. The main challenge is to be cognizant of the clinical syndromes and to be able to differentiate diseases caused by bioterrorism from naturally occurring disorders. This review provides an overview of the biological and chemical warfare agents, with a focus on neurological manifestation and an approach to treatment from a perspective of neurological critical care. ELECTRONIC SUPPLEMENTARY MATERIAL: The online version of this article (doi:10.1007/s13311-011-0097-2) contains supplementary material, which is available to authorized users.',\n",
       "     'journal': 'Neurotherapeutics',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC3271146',\n",
       "     'authors': 'Busl, Katharina M.; Bleck, Thomas P.',\n",
       "     's2_id': None,\n",
       "     'pubmed_id': '22227729',\n",
       "     'publish_time': '2012-01-07',\n",
       "     'doi': '10.1007/s13311-011-0097-2',\n",
       "     'message': 'sgcf9v3m,2a753c9ec1b4376606a18b951f4257a602248ddd,PMC,Treatment of Neuroterrorism,10.1007/s13311-011-0097-2,PMC3271146,22227729,no-cc,\"Bioterrorism is defined as the intentional use of biological, chemical, nuclear, or radiological agents to cause disease, death, or environmental damage. Early recognition of a bioterrorist attack is of utmost importance to minimize casualties and initiate appropriate therapy. The range of agents that could potentially be used as weapons is wide, however, only a few of these agents have all the characteristics making them ideal for that purpose. Many of the chemical and biological weapons can cause neurological symptoms and damage the nervous system in varying degrees. Therefore, preparedness among neurologists is important. The main challenge is to be cognizant of the clinical syndromes and to be able to differentiate diseases caused by bioterrorism from naturally occurring disorders. This review provides an overview of the biological and chemical warfare agents, with a focus on neurological manifestation and an approach to treatment from a perspective of neurological critical care. ELECTRONIC SUPPLEMENTARY MATERIAL: The online version of this article (doi:10.1007/s13311-011-0097-2) contains supplementary material, which is available to authorized users.\",2012-01-07,\"Busl, Katharina M.; Bleck, Thomas P.\",Neurotherapeutics,,,,document_parses/pdf_json/2a753c9ec1b4376606a18b951f4257a602248ddd.json,document_parses/pmc_json/PMC3271146.xml.json,https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3271146/,\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'TzhW93MBdq74P7QnRv-0',\n",
       "    '_score': 1.0,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:54:37.673Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'cc-by',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'New Cardiovascular and Pulmonary Therapeutic Strategies Based on the Angiotensin-Converting Enzyme 2/Angiotensin-(1–7)/Mas Receptor Axis',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': '880b3f24a1f3678dd29cb446dae24298863b8676',\n",
       "     'cord_uid': '0qkzd2w4',\n",
       "     'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3272817/',\n",
       "     'source_x': 'PMC',\n",
       "     'pmc_json_files': 'document_parses/pmc_json/PMC3272817.xml.json',\n",
       "     'pdf_json_files': 'document_parses/pdf_json/880b3f24a1f3678dd29cb446dae24298863b8676.json',\n",
       "     'abstract': 'Angiotensin (Ang)-(1–7) is now recognized as a biologically active component of the renin-angiotensin system (RAS). The discovery of the angiotensin-converting enzyme homologue ACE2 revealed important metabolic pathways involved in the Ang-(1–7) synthesis. This enzyme can form Ang-(1–7) from Ang II or less efficiently through hydrolysis of Ang I to Ang-(1–9) with subsequent Ang-(1–7) formation. Additionally, it is well established that the G protein-coupled receptor Mas is a functional ligand site for Ang-(1–7). The axis formed by ACE2/Ang-(1–7)/Mas represents an endogenous counter regulatory pathway within the RAS whose actions are opposite to the vasoconstrictor/proliferative arm of the RAS constituted by ACE/Ang II/AT(1) receptor. In this review we will discuss recent findings concerning the biological role of the ACE2/Ang-(1–7)/Mas arm in the cardiovascular and pulmonary system. Also, we will highlight the initiatives to develop potential therapeutic strategies based on this axis.',\n",
       "     'journal': 'Int J Hypertens',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC3272817',\n",
       "     'authors': 'Ferreira, Anderson J.; Murça, Tatiane M.; Fraga-Silva, Rodrigo A.; Castro, Carlos Henrique; Raizada, Mohan K.; Santos, Robson A. S.',\n",
       "     's2_id': None,\n",
       "     'pubmed_id': '22319643',\n",
       "     'publish_time': '2012-01-26',\n",
       "     'doi': '10.1155/2012/147825',\n",
       "     'message': '0qkzd2w4,880b3f24a1f3678dd29cb446dae24298863b8676,PMC,New Cardiovascular and Pulmonary Therapeutic Strategies Based on the Angiotensin-Converting Enzyme 2/Angiotensin-(1–7)/Mas Receptor Axis,10.1155/2012/147825,PMC3272817,22319643,cc-by,\"Angiotensin (Ang)-(1–7) is now recognized as a biologically active component of the renin-angiotensin system (RAS). The discovery of the angiotensin-converting enzyme homologue ACE2 revealed important metabolic pathways involved in the Ang-(1–7) synthesis. This enzyme can form Ang-(1–7) from Ang II or less efficiently through hydrolysis of Ang I to Ang-(1–9) with subsequent Ang-(1–7) formation. Additionally, it is well established that the G protein-coupled receptor Mas is a functional ligand site for Ang-(1–7). The axis formed by ACE2/Ang-(1–7)/Mas represents an endogenous counter regulatory pathway within the RAS whose actions are opposite to the vasoconstrictor/proliferative arm of the RAS constituted by ACE/Ang II/AT(1) receptor. In this review we will discuss recent findings concerning the biological role of the ACE2/Ang-(1–7)/Mas arm in the cardiovascular and pulmonary system. Also, we will highlight the initiatives to develop potential therapeutic strategies based on this axis.\",2012-01-26,\"Ferreira, Anderson J.; Murça, Tatiane M.; Fraga-Silva, Rodrigo A.; Castro, Carlos Henrique; Raizada, Mohan K.; Santos, Robson A. S.\",Int J Hypertens,,,,document_parses/pdf_json/880b3f24a1f3678dd29cb446dae24298863b8676.json,document_parses/pmc_json/PMC3272817.xml.json,https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3272817/,\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'UDhW93MBdq74P7QnRv-0',\n",
       "    '_score': 1.0,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:54:37.674Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'cc-by-nc',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Unraveling the structural complexity in a single-stranded RNA tail: implications for efficient ligand binding in the prequeuosine riboswitch',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': 'd1268a2b5aa5778f15fbd10152728dc936a6809e',\n",
       "     'cord_uid': 'kci1lkhj',\n",
       "     'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3273816/',\n",
       "     'source_x': 'PMC',\n",
       "     'pmc_json_files': 'document_parses/pmc_json/PMC3273816.xml.json',\n",
       "     'pdf_json_files': 'document_parses/pdf_json/d1268a2b5aa5778f15fbd10152728dc936a6809e.json',\n",
       "     'abstract': 'Single-stranded RNAs (ssRNAs) are ubiquitous RNA elements that serve diverse functional roles. Much of our understanding of ssRNA conformational behavior is limited to structures in which ssRNA directly engages in tertiary interactions or is recognized by proteins. Little is known about the structural and dynamic behavior of free ssRNAs at atomic resolution. Here, we report the collaborative application of nuclear magnetic resonance (NMR) and replica exchange molecular dynamics (REMD) simulations to characterize the 12 nt ssRNA tail derived from the prequeuosine riboswitch. NMR carbon spin relaxation data and residual dipolar coupling measurements reveal a flexible yet stacked core adopting an A-form-like conformation, with the level of order decreasing toward the terminal ends. An A-to-C mutation within the polyadenine tract alters the observed dynamics consistent with the introduction of a dynamic kink. Pre-ordering of the tail may increase the efficacy of ligand binding above that achieved by a random-coil ssRNA. The REMD simulations recapitulate important trends in the NMR data, but suggest more internal motions than inferred from the NMR analysis. Our study unmasks a previously unappreciated level of complexity in ssRNA, which we believe will also serve as an excellent model system for testing and developing computational force fields.',\n",
       "     'journal': 'Nucleic Acids Res',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC3273816',\n",
       "     'authors': 'Eichhorn, Catherine D.; Feng, Jun; Suddala, Krishna C.; Walter, Nils G.; Brooks, Charles L.; Al-Hashimi, Hashim M.',\n",
       "     's2_id': None,\n",
       "     'pubmed_id': '22009676',\n",
       "     'publish_time': '2011-10-18',\n",
       "     'doi': '10.1093/nar/gkr833',\n",
       "     'message': 'kci1lkhj,d1268a2b5aa5778f15fbd10152728dc936a6809e,PMC,Unraveling the structural complexity in a single-stranded RNA tail: implications for efficient ligand binding in the prequeuosine riboswitch,10.1093/nar/gkr833,PMC3273816,22009676,cc-by-nc,\"Single-stranded RNAs (ssRNAs) are ubiquitous RNA elements that serve diverse functional roles. Much of our understanding of ssRNA conformational behavior is limited to structures in which ssRNA directly engages in tertiary interactions or is recognized by proteins. Little is known about the structural and dynamic behavior of free ssRNAs at atomic resolution. Here, we report the collaborative application of nuclear magnetic resonance (NMR) and replica exchange molecular dynamics (REMD) simulations to characterize the 12 nt ssRNA tail derived from the prequeuosine riboswitch. NMR carbon spin relaxation data and residual dipolar coupling measurements reveal a flexible yet stacked core adopting an A-form-like conformation, with the level of order decreasing toward the terminal ends. An A-to-C mutation within the polyadenine tract alters the observed dynamics consistent with the introduction of a dynamic kink. Pre-ordering of the tail may increase the efficacy of ligand binding above that achieved by a random-coil ssRNA. The REMD simulations recapitulate important trends in the NMR data, but suggest more internal motions than inferred from the NMR analysis. Our study unmasks a previously unappreciated level of complexity in ssRNA, which we believe will also serve as an excellent model system for testing and developing computational force fields.\",2011-10-18,\"Eichhorn, Catherine D.; Feng, Jun; Suddala, Krishna C.; Walter, Nils G.; Brooks, Charles L.; Al-Hashimi, Hashim M.\",Nucleic Acids Res,,,,document_parses/pdf_json/d1268a2b5aa5778f15fbd10152728dc936a6809e.json,document_parses/pmc_json/PMC3273816.xml.json,https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3273816/,\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'UThW93MBdq74P7QnRv-0',\n",
       "    '_score': 1.0,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:54:37.674Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'cc-by',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Soluble RAGE as a severity marker in community acquired pneumonia associated sepsis',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': '05bbb228a1d50e89500669fec042d288d0796c6b',\n",
       "     'cord_uid': 'abml6wgd',\n",
       "     'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3274434/',\n",
       "     'source_x': 'PMC',\n",
       "     'pmc_json_files': 'document_parses/pmc_json/PMC3274434.xml.json',\n",
       "     'pdf_json_files': 'document_parses/pdf_json/05bbb228a1d50e89500669fec042d288d0796c6b.json',\n",
       "     'abstract': 'BACKGROUND: Community-acquired pneumonia (CAP) is considered the most important cause of death from infectious disease in developed countries. Severity assessment scores partially address the difficulties in identifying high-risk patients. A lack of specific and valid pathophysiologic severity markers affect early and effective sepsis therapy. HMGB-1, sRAGE and RAGE have been involved in sepsis and their potential as severity markers has been proposed. The aim of this study was to evaluate HMGB-1, RAGE and sRAGE levels in patients with CAP-associated sepsis and determine their possible association with clinical outcome. METHOD: We evaluated 33 patients with CAP-associated sepsis admitted to the emergency room and followed in the medical wards. Severity assessment scores (CURB-65, PSI, APACHE II, SOFA) and serologic markers (HMGB-1, RAGE, sRAGE) were evaluated on admission. RESULTS: Thirty patients with a diagnosis of CAP-associated sepsis were enrolled in the study within 24 hours after admission. Fourteen (46.6%) had pandemic (H1N1) influenza A virus, 2 (6.6%) had seasonal influenza A and 14 other diagnoses. Of the patients in the study group, 16 (53.3%) had a fatal outcome. ARDS was observed in 17 (56.6%) and a total of 22 patients had severe sepsis on admission (73%). The SOFA score showed the greatest difference between surviving and non-surviving groups (P = .003) with similar results in ARDS patients (P = .005). sRAGE levels tended to be higher in non-surviving (P = .058) and ARDS patients (P = .058). Logistic regression modeling demonstrated that SOFA (P = .013) and sRAGE (P = .05) were the only variables that modified the probability of a fatal outcome. CONCLUSION: The association of elevated sRAGE with a fatal outcome suggests that it may have an independent causal effect in CAP. SOFA scores were the only clinical factor with the ability to identify surviving and ARDS patients.',\n",
       "     'journal': 'BMC Infect Dis',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC3274434',\n",
       "     'authors': 'Narvaez-Rivera, Rodrigo M; Rendon, Adrian; Salinas-Carmona, Mario C; Rosas-Taraco, Adrian G',\n",
       "     's2_id': None,\n",
       "     'pubmed_id': '22264245',\n",
       "     'publish_time': '2012-01-20',\n",
       "     'doi': '10.1186/1471-2334-12-15',\n",
       "     'message': 'abml6wgd,05bbb228a1d50e89500669fec042d288d0796c6b,PMC,Soluble RAGE as a severity marker in community acquired pneumonia associated sepsis,10.1186/1471-2334-12-15,PMC3274434,22264245,cc-by,\"BACKGROUND: Community-acquired pneumonia (CAP) is considered the most important cause of death from infectious disease in developed countries. Severity assessment scores partially address the difficulties in identifying high-risk patients. A lack of specific and valid pathophysiologic severity markers affect early and effective sepsis therapy. HMGB-1, sRAGE and RAGE have been involved in sepsis and their potential as severity markers has been proposed. The aim of this study was to evaluate HMGB-1, RAGE and sRAGE levels in patients with CAP-associated sepsis and determine their possible association with clinical outcome. METHOD: We evaluated 33 patients with CAP-associated sepsis admitted to the emergency room and followed in the medical wards. Severity assessment scores (CURB-65, PSI, APACHE II, SOFA) and serologic markers (HMGB-1, RAGE, sRAGE) were evaluated on admission. RESULTS: Thirty patients with a diagnosis of CAP-associated sepsis were enrolled in the study within 24 hours after admission. Fourteen (46.6%) had pandemic (H1N1) influenza A virus, 2 (6.6%) had seasonal influenza A and 14 other diagnoses. Of the patients in the study group, 16 (53.3%) had a fatal outcome. ARDS was observed in 17 (56.6%) and a total of 22 patients had severe sepsis on admission (73%). The SOFA score showed the greatest difference between surviving and non-surviving groups (P = .003) with similar results in ARDS patients (P = .005). sRAGE levels tended to be higher in non-surviving (P = .058) and ARDS patients (P = .058). Logistic regression modeling demonstrated that SOFA (P = .013) and sRAGE (P = .05) were the only variables that modified the probability of a fatal outcome. CONCLUSION: The association of elevated sRAGE with a fatal outcome suggests that it may have an independent causal effect in CAP. SOFA scores were the only clinical factor with the ability to identify surviving and ARDS patients.\",2012-01-20,\"Narvaez-Rivera, Rodrigo M; Rendon, Adrian; Salinas-Carmona, Mario C; Rosas-Taraco, Adrian G\",BMC Infect Dis,,,,document_parses/pdf_json/05bbb228a1d50e89500669fec042d288d0796c6b.json,document_parses/pmc_json/PMC3274434.xml.json,https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3274434/,\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'UjhW93MBdq74P7QnRv-0',\n",
       "    '_score': 1.0,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:54:37.674Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'cc-by',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Virus Identification in Unknown Tropical Febrile Illness Cases Using Deep Sequencing',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': '9692bb55e1e2eec083333ee2139137e6ddf3a4d8',\n",
       "     'cord_uid': '6gow0x1v',\n",
       "     'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3274504/',\n",
       "     'source_x': 'PMC',\n",
       "     'pmc_json_files': 'document_parses/pmc_json/PMC3274504.xml.json',\n",
       "     'pdf_json_files': 'document_parses/pdf_json/9692bb55e1e2eec083333ee2139137e6ddf3a4d8.json',\n",
       "     'abstract': 'Dengue virus is an emerging infectious agent that infects an estimated 50–100 million people annually worldwide, yet current diagnostic practices cannot detect an etiologic pathogen in ∼40% of dengue-like illnesses. Metagenomic approaches to pathogen detection, such as viral microarrays and deep sequencing, are promising tools to address emerging and non-diagnosable disease challenges. In this study, we used the Virochip microarray and deep sequencing to characterize the spectrum of viruses present in human sera from 123 Nicaraguan patients presenting with dengue-like symptoms but testing negative for dengue virus. We utilized a barcoding strategy to simultaneously deep sequence multiple serum specimens, generating on average over 1 million reads per sample. We then implemented a stepwise bioinformatic filtering pipeline to remove the majority of human and low-quality sequences to improve the speed and accuracy of subsequent unbiased database searches. By deep sequencing, we were able to detect virus sequence in 37% (45/123) of previously negative cases. These included 13 cases with Human Herpesvirus 6 sequences. Other samples contained sequences with similarity to sequences from viruses in the Herpesviridae, Flaviviridae, Circoviridae, Anelloviridae, Asfarviridae, and Parvoviridae families. In some cases, the putative viral sequences were virtually identical to known viruses, and in others they diverged, suggesting that they may derive from novel viruses. These results demonstrate the utility of unbiased metagenomic approaches in the detection of known and divergent viruses in the study of tropical febrile illness.',\n",
       "     'journal': 'PLoS Negl Trop Dis',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC3274504',\n",
       "     'authors': 'Yozwiak, Nathan L.; Skewes-Cox, Peter; Stenglein, Mark D.; Balmaseda, Angel; Harris, Eva; DeRisi, Joseph L.',\n",
       "     's2_id': None,\n",
       "     'pubmed_id': '22347512',\n",
       "     'publish_time': '2012-02-07',\n",
       "     'doi': '10.1371/journal.pntd.0001485',\n",
       "     'message': '6gow0x1v,9692bb55e1e2eec083333ee2139137e6ddf3a4d8,PMC,Virus Identification in Unknown Tropical Febrile Illness Cases Using Deep Sequencing,10.1371/journal.pntd.0001485,PMC3274504,22347512,cc-by,\"Dengue virus is an emerging infectious agent that infects an estimated 50–100 million people annually worldwide, yet current diagnostic practices cannot detect an etiologic pathogen in ∼40% of dengue-like illnesses. Metagenomic approaches to pathogen detection, such as viral microarrays and deep sequencing, are promising tools to address emerging and non-diagnosable disease challenges. In this study, we used the Virochip microarray and deep sequencing to characterize the spectrum of viruses present in human sera from 123 Nicaraguan patients presenting with dengue-like symptoms but testing negative for dengue virus. We utilized a barcoding strategy to simultaneously deep sequence multiple serum specimens, generating on average over 1 million reads per sample. We then implemented a stepwise bioinformatic filtering pipeline to remove the majority of human and low-quality sequences to improve the speed and accuracy of subsequent unbiased database searches. By deep sequencing, we were able to detect virus sequence in 37% (45/123) of previously negative cases. These included 13 cases with Human Herpesvirus 6 sequences. Other samples contained sequences with similarity to sequences from viruses in the Herpesviridae, Flaviviridae, Circoviridae, Anelloviridae, Asfarviridae, and Parvoviridae families. In some cases, the putative viral sequences were virtually identical to known viruses, and in others they diverged, suggesting that they may derive from novel viruses. These results demonstrate the utility of unbiased metagenomic approaches in the detection of known and divergent viruses in the study of tropical febrile illness.\",2012-02-07,\"Yozwiak, Nathan L.; Skewes-Cox, Peter; Stenglein, Mark D.; Balmaseda, Angel; Harris, Eva; DeRisi, Joseph L.\",PLoS Negl Trop Dis,,,,document_parses/pdf_json/9692bb55e1e2eec083333ee2139137e6ddf3a4d8.json,document_parses/pmc_json/PMC3274504.xml.json,https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3274504/,\\r'}}]}}"
      ]
     },
     "execution_count": 32,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "x = es.search(index = \"metafile\")\n",
    "x"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## What is known about transmission, incubation, and environmental stability?"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Range of incubation periods for the disease in humans (and how this varies across age and health status) and\n",
    "#how long individuals are contagious, even after recovery.\n",
    "incubation = es.search(index = \"metafile\", body= {\"query\": { \"match\": {\"title\": \"incubation periods for the disease in humans\"}}})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'took': 444,\n",
       " 'timed_out': False,\n",
       " '_shards': {'total': 1, 'successful': 1, 'skipped': 0, 'failed': 0},\n",
       " 'hits': {'total': {'value': 10000, 'relation': 'gte'},\n",
       "  'max_score': 19.853497,\n",
       "  'hits': [{'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'HDta93MBdq74P7QnqKD2',\n",
       "    '_score': 19.853497,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:59:40.075Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'no-cc',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Multiple Contact Dates and SARS Incubation Periods',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': '24de65541e25ca372d5fb686b733b45a1dd5c0b6',\n",
       "     'cord_uid': '13xmmin1',\n",
       "     'url': 'https://www.ncbi.nlm.nih.gov/pubmed/15030684/',\n",
       "     'source_x': 'Medline; PMC',\n",
       "     'pmc_json_files': 'document_parses/pmc_json/PMC3322923.xml.json',\n",
       "     'pdf_json_files': 'document_parses/pdf_json/24de65541e25ca372d5fb686b733b45a1dd5c0b6.json',\n",
       "     'abstract': 'Many severe acute respiratory syndrome (SARS) patients have multiple possible incubation periods due to multiple contact dates. Multiple contact dates cannot be used in standard statistical analytic techniques, however. I present a simple spreadsheet-based method that uses multiple contact dates to calculate the possible incubation periods of SARS.',\n",
       "     'journal': 'Emerg Infect Dis',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC3322923',\n",
       "     'authors': 'Meltzer, Martin I.',\n",
       "     's2_id': '12741708',\n",
       "     'pubmed_id': '15030684',\n",
       "     'publish_time': '2004-02-29',\n",
       "     'doi': '10.3201/eid1002.030426',\n",
       "     'message': '13xmmin1,24de65541e25ca372d5fb686b733b45a1dd5c0b6,Medline; PMC,Multiple Contact Dates and SARS Incubation Periods,10.3201/eid1002.030426,PMC3322923,15030684,no-cc,\"Many severe acute respiratory syndrome (SARS) patients have multiple possible incubation periods due to multiple contact dates. Multiple contact dates cannot be used in standard statistical analytic techniques, however. I present a simple spreadsheet-based method that uses multiple contact dates to calculate the possible incubation periods of SARS.\",2004-02-29,\"Meltzer, Martin I.\",Emerg Infect Dis,,,,document_parses/pdf_json/24de65541e25ca372d5fb686b733b45a1dd5c0b6.json,document_parses/pmc_json/PMC3322923.xml.json,https://www.ncbi.nlm.nih.gov/pubmed/15030684/,12741708\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'iDlW93MBdq74P7QnZAFm',\n",
       "    '_score': 19.134518,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:54:52.558Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'cc-by',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Incubation periods of viral gastroenteritis: a systematic review',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': '599f44a88bfd9fcd7cc5b03f3b0bf01c9b3c5ba8',\n",
       "     'cord_uid': '553cjp9f',\n",
       "     'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3849296/',\n",
       "     'source_x': 'PMC',\n",
       "     'pmc_json_files': 'document_parses/pmc_json/PMC3849296.xml.json',\n",
       "     'pdf_json_files': 'document_parses/pdf_json/599f44a88bfd9fcd7cc5b03f3b0bf01c9b3c5ba8.json',\n",
       "     'abstract': 'BACKGROUND: Accurate knowledge of incubation period is important to investigate and to control infectious diseases and their transmission, however statements of incubation period in the literature are often uncited, inconsistent, and/or not evidence based. METHODS: In a systematic review of the literature on five enteric viruses of public health importance, we found 256 articles with incubation period estimates, including 33 with data for pooled analysis. RESULTS: We fit a log-normal distribution to pooled data and found the median incubation period to be 4.5 days (95% CI 3.9-5.2 days) for astrovirus, 1.2 days (95% CI 1.1-1.2 days) for norovirus genogroups I and II, 1.7 days (95% CI 1.5-1.8 days) for sapovirus, and 2.0 days (95% CI 1.4-2.4 days) for rotavirus. CONCLUSIONS: Our estimates combine published data and provide sufficient quantitative detail to allow for these estimates to be used in a wide range of clinical and modeling applications. This can translate into improved prevention and control efforts in settings with transmission or the risk of transmission.',\n",
       "     'journal': 'BMC Infect Dis',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC3849296',\n",
       "     'authors': 'Lee, Rachel M; Lessler, Justin; Lee, Rose A; Rudolph, Kara E; Reich, Nicholas G; Perl, Trish M; Cummings, Derek AT',\n",
       "     's2_id': None,\n",
       "     'pubmed_id': '24066865',\n",
       "     'publish_time': '2013-09-25',\n",
       "     'doi': '10.1186/1471-2334-13-446',\n",
       "     'message': '553cjp9f,599f44a88bfd9fcd7cc5b03f3b0bf01c9b3c5ba8,PMC,Incubation periods of viral gastroenteritis: a systematic review,10.1186/1471-2334-13-446,PMC3849296,24066865,cc-by,\"BACKGROUND: Accurate knowledge of incubation period is important to investigate and to control infectious diseases and their transmission, however statements of incubation period in the literature are often uncited, inconsistent, and/or not evidence based. METHODS: In a systematic review of the literature on five enteric viruses of public health importance, we found 256 articles with incubation period estimates, including 33 with data for pooled analysis. RESULTS: We fit a log-normal distribution to pooled data and found the median incubation period to be 4.5 days (95% CI 3.9-5.2 days) for astrovirus, 1.2 days (95% CI 1.1-1.2 days) for norovirus genogroups I and II, 1.7 days (95% CI 1.5-1.8 days) for sapovirus, and 2.0 days (95% CI 1.4-2.4 days) for rotavirus. CONCLUSIONS: Our estimates combine published data and provide sufficient quantitative detail to allow for these estimates to be used in a wide range of clinical and modeling applications. This can translate into improved prevention and control efforts in settings with transmission or the risk of transmission.\",2013-09-25,\"Lee, Rachel M; Lessler, Justin; Lee, Rose A; Rudolph, Kara E; Reich, Nicholas G; Perl, Trish M; Cummings, Derek AT\",BMC Infect Dis,,,,document_parses/pdf_json/599f44a88bfd9fcd7cc5b03f3b0bf01c9b3c5ba8.json,document_parses/pmc_json/PMC3849296.xml.json,https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3849296/,\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'wzta93MBdq74P7Qn6d_a',\n",
       "    '_score': 18.878075,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:59:56.644Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'cc-by',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Visualizing Clinical Evidence: Citation Networks for the Incubation Periods of Respiratory Viral Infections',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': '7d84b116132df661fef2f697880e3856e1e4ca8a; 21e6d4c42e4375fcb05eeebb140a805461688542; cfffac30aa716974333312a44475097d94c8f475',\n",
       "     'cord_uid': '57av5x3o',\n",
       "     'url': 'https://doi.org/10.1371/journal.pone.0019496; https://www.ncbi.nlm.nih.gov/pubmed/21559339/',\n",
       "     'source_x': 'Medline; PMC',\n",
       "     'pmc_json_files': 'document_parses/pmc_json/PMC3084881.xml.json',\n",
       "     'pdf_json_files': 'document_parses/pdf_json/7d84b116132df661fef2f697880e3856e1e4ca8a.json; document_parses/pdf_json/21e6d4c42e4375fcb05eeebb140a805461688542.json; document_parses/pdf_json/cfffac30aa716974333312a44475097d94c8f475.json',\n",
       "     'abstract': 'Simply by repetition, medical facts can become enshrined as truth even when there is little empirical evidence supporting them. We present an intuitive and clear visual design for tracking the citation history of a particular scientific fact over time. We apply this method to data from a previously published literature review on the incubation period of nine respiratory viral infections. The resulting citation networks reveal that the conventional wisdom about the incubation period for these diseases was based on a small fraction of available data and in one case, on no retrievable empirical evidence. Overall, 50% of all incubation period statements did not provide a source for their estimate and 65% of original sources for incubation period data were not incorporated into subsequent publications. More standardized and widely available methods for visualizing these histories of medical evidence are needed to ensure that conventional wisdom cannot stray too far from empirically supported knowledge.',\n",
       "     'journal': 'PLoS One',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC3084881',\n",
       "     'authors': 'Reich, Nicholas G.; Perl, Trish M.; Cummings, Derek A. T.; Lessler, Justin',\n",
       "     's2_id': '16370601',\n",
       "     'pubmed_id': '21559339',\n",
       "     'publish_time': '2011-04-29',\n",
       "     'doi': '10.1371/journal.pone.0019496',\n",
       "     'message': '57av5x3o,7d84b116132df661fef2f697880e3856e1e4ca8a; 21e6d4c42e4375fcb05eeebb140a805461688542; cfffac30aa716974333312a44475097d94c8f475,Medline; PMC,Visualizing Clinical Evidence: Citation Networks for the Incubation Periods of Respiratory Viral Infections,10.1371/journal.pone.0019496,PMC3084881,21559339,cc-by,\"Simply by repetition, medical facts can become enshrined as truth even when there is little empirical evidence supporting them. We present an intuitive and clear visual design for tracking the citation history of a particular scientific fact over time. We apply this method to data from a previously published literature review on the incubation period of nine respiratory viral infections. The resulting citation networks reveal that the conventional wisdom about the incubation period for these diseases was based on a small fraction of available data and in one case, on no retrievable empirical evidence. Overall, 50% of all incubation period statements did not provide a source for their estimate and 65% of original sources for incubation period data were not incorporated into subsequent publications. More standardized and widely available methods for visualizing these histories of medical evidence are needed to ensure that conventional wisdom cannot stray too far from empirically supported knowledge.\",2011-04-29,\"Reich, Nicholas G.; Perl, Trish M.; Cummings, Derek A. T.; Lessler, Justin\",PLoS One,,,,document_parses/pdf_json/7d84b116132df661fef2f697880e3856e1e4ca8a.json; document_parses/pdf_json/21e6d4c42e4375fcb05eeebb140a805461688542.json; document_parses/pdf_json/cfffac30aa716974333312a44475097d94c8f475.json,document_parses/pmc_json/PMC3084881.xml.json,https://doi.org/10.1371/journal.pone.0019496; https://www.ncbi.nlm.nih.gov/pubmed/21559339/,16370601\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'Hzta93MBdq74P7QnwLf1',\n",
       "    '_score': 17.842234,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:59:46.377Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'green-oa',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Incubation periods of acute respiratory viral infections: a systematic review',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': '38626725ce89692017cd5c7b529da381e1412b11; 55f45cef2d7df64aa4f3d5d80e554030617b7b59; 31e1b495e358bdfb9ac295b4e1cdd8052056499d',\n",
       "     'cord_uid': 'iicbaevw',\n",
       "     'url': 'https://doi.org/10.1016/s1473-3099(09)70069-6; https://www.sciencedirect.com/science/article/pii/S1473309909700696; https://www.ncbi.nlm.nih.gov/pubmed/19393959/; http://europepmc.org/articles/pmc4327893?pdf=render; https://api.elsevier.com/content/article/pii/S1473309909700696',\n",
       "     'source_x': 'Elsevier; Medline; PMC',\n",
       "     'pmc_json_files': 'document_parses/pmc_json/PMC4327893.xml.json',\n",
       "     'pdf_json_files': 'document_parses/pdf_json/38626725ce89692017cd5c7b529da381e1412b11.json; document_parses/pdf_json/55f45cef2d7df64aa4f3d5d80e554030617b7b59.json; document_parses/pdf_json/31e1b495e358bdfb9ac295b4e1cdd8052056499d.json',\n",
       "     'abstract': 'Knowledge of the incubation period is essential in the investigation and control of infectious disease, but statements of incubation period are often poorly referenced, inconsistent, or based on limited data. In a systematic review of the literature on nine respiratory viral infections of public-health importance, we identified 436 articles with statements of incubation period and 38 with data for pooled analysis. We fitted a log-normal distribution to pooled data and found the median incubation period to be 5·6 days (95% CI 4·8–6·3) for adenovirus, 3·2 days (95% CI 2·8–3·7) for human coronavirus, 4·0 days (95% CI 3·6–4·4) for severe acute respiratory syndrome coronavirus, 1·4 days (95% CI 1·3–1·5) for influenza A, 0·6 days (95% CI 0·5–0·6) for influenza B, 12·5 days (95% CI 11·8–13·3) for measles, 2·6 days (95% CI 2·1–3·1) for parainfluenza, 4·4 days (95% CI 3·9–4·9) for respiratory syncytial virus, and 1·9 days (95% CI 1·4–2·4) for rhinovirus. When using the incubation period, it is important to consider its full distribution: the right tail for quarantine policy, the central regions for likely times and sources of infection, and the full distribution for models used in pandemic planning. Our estimates combine published data to give the detail necessary for these and other applications.',\n",
       "     'journal': 'The Lancet Infectious Diseases',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC4327893',\n",
       "     'authors': 'Lessler, Justin; Reich, Nicholas G; Brookmeyer, Ron; Perl, Trish M; Nelson, Kenrad E; Cummings, Derek A T',\n",
       "     's2_id': '1794905',\n",
       "     'pubmed_id': '19393959',\n",
       "     'publish_time': '2009-05-01',\n",
       "     'doi': '10.1016/s1473-3099(09)70069-6',\n",
       "     'message': 'iicbaevw,38626725ce89692017cd5c7b529da381e1412b11; 55f45cef2d7df64aa4f3d5d80e554030617b7b59; 31e1b495e358bdfb9ac295b4e1cdd8052056499d,Elsevier; Medline; PMC,Incubation periods of acute respiratory viral infections: a systematic review,10.1016/s1473-3099(09)70069-6,PMC4327893,19393959,green-oa,\"Knowledge of the incubation period is essential in the investigation and control of infectious disease, but statements of incubation period are often poorly referenced, inconsistent, or based on limited data. In a systematic review of the literature on nine respiratory viral infections of public-health importance, we identified 436 articles with statements of incubation period and 38 with data for pooled analysis. We fitted a log-normal distribution to pooled data and found the median incubation period to be 5·6 days (95% CI 4·8–6·3) for adenovirus, 3·2 days (95% CI 2·8–3·7) for human coronavirus, 4·0 days (95% CI 3·6–4·4) for severe acute respiratory syndrome coronavirus, 1·4 days (95% CI 1·3–1·5) for influenza A, 0·6 days (95% CI 0·5–0·6) for influenza B, 12·5 days (95% CI 11·8–13·3) for measles, 2·6 days (95% CI 2·1–3·1) for parainfluenza, 4·4 days (95% CI 3·9–4·9) for respiratory syncytial virus, and 1·9 days (95% CI 1·4–2·4) for rhinovirus. When using the incubation period, it is important to consider its full distribution: the right tail for quarantine policy, the central regions for likely times and sources of infection, and the full distribution for models used in pandemic planning. Our estimates combine published data to give the detail necessary for these and other applications.\",2009-05-01,\"Lessler, Justin; Reich, Nicholas G; Brookmeyer, Ron; Perl, Trish M; Nelson, Kenrad E; Cummings, Derek A T\",The Lancet Infectious Diseases,,,,document_parses/pdf_json/38626725ce89692017cd5c7b529da381e1412b11.json; document_parses/pdf_json/55f45cef2d7df64aa4f3d5d80e554030617b7b59.json; document_parses/pdf_json/31e1b495e358bdfb9ac295b4e1cdd8052056499d.json,document_parses/pmc_json/PMC4327893.xml.json,https://doi.org/10.1016/s1473-3099(09)70069-6; https://www.sciencedirect.com/science/article/pii/S1473309909700696; https://www.ncbi.nlm.nih.gov/pubmed/19393959/; http://europepmc.org/articles/pmc4327893?pdf=render; https://api.elsevier.com/content/article/pii/S1473309909700696,1794905\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': '6zlW93MBdq74P7QnhQfa',\n",
       "    '_score': 12.631602,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:55:07.230Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'cc-by',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Estimating the incubation period of hand, foot and mouth disease for children in different age groups',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': '9f63dea0f76ee477d2e8e5209d40179db431ab1d',\n",
       "     'cord_uid': 'gkvmjozy',\n",
       "     'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5705633/',\n",
       "     'source_x': 'PMC',\n",
       "     'pmc_json_files': 'document_parses/pmc_json/PMC5705633.xml.json',\n",
       "     'pdf_json_files': 'document_parses/pdf_json/9f63dea0f76ee477d2e8e5209d40179db431ab1d.json',\n",
       "     'abstract': 'Hand, foot and mouth disease (HFMD) is a childhood disease causing large outbreaks frequently in Asia and occasionally in Europe and the US. The incubation period of HFMD was typically described as about 3–7 days but empirical evidence is lacking. In this study, we estimated the incubation period of HFMD from school outbreaks in Hong Kong, utilizing information on symptom onset and sick absence dates of students diagnosed with HFMD. A total of 99 HFMD cases from 12 schools were selected for analysis. We fitted parametric models accounting for interval censoring. Based on the best-fitted distributions, the estimated median incubation periods were 4.4 (95% CI 3.8–5.1) days, 4.7 (95% CI 4.5–5.1) days and 5.7 (95% CI 4.6–7.0) days for children in kindergartens, primary schools and secondary schools respectively. From the fitted distribution, the estimated incubation periods can be longer than 10 days for 8.8% and 23.2% of the HFMD cases in kindergarten and secondary schools respectively. Our results show that the incubation period of HFMD for secondary schools students can be longer than the ranges commonly described. An extended period of enhanced personal hygiene practice and disinfection of the environment may be needed to control outbreaks.',\n",
       "     'journal': 'Sci Rep',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC5705633',\n",
       "     'authors': 'Yang, Zhongzhou; Zhang, Qiqi; Cowling, Benjamin J.; Lau, Eric H. Y.',\n",
       "     's2_id': None,\n",
       "     'pubmed_id': '29184105',\n",
       "     'publish_time': '2017-11-28',\n",
       "     'doi': '10.1038/s41598-017-16705-7',\n",
       "     'message': 'gkvmjozy,9f63dea0f76ee477d2e8e5209d40179db431ab1d,PMC,\"Estimating the incubation period of hand, foot and mouth disease for children in different age groups\",10.1038/s41598-017-16705-7,PMC5705633,29184105,cc-by,\"Hand, foot and mouth disease (HFMD) is a childhood disease causing large outbreaks frequently in Asia and occasionally in Europe and the US. The incubation period of HFMD was typically described as about 3–7 days but empirical evidence is lacking. In this study, we estimated the incubation period of HFMD from school outbreaks in Hong Kong, utilizing information on symptom onset and sick absence dates of students diagnosed with HFMD. A total of 99 HFMD cases from 12 schools were selected for analysis. We fitted parametric models accounting for interval censoring. Based on the best-fitted distributions, the estimated median incubation periods were 4.4 (95% CI 3.8–5.1) days, 4.7 (95% CI 4.5–5.1) days and 5.7 (95% CI 4.6–7.0) days for children in kindergartens, primary schools and secondary schools respectively. From the fitted distribution, the estimated incubation periods can be longer than 10 days for 8.8% and 23.2% of the HFMD cases in kindergarten and secondary schools respectively. Our results show that the incubation period of HFMD for secondary schools students can be longer than the ranges commonly described. An extended period of enhanced personal hygiene practice and disinfection of the environment may be needed to control outbreaks.\",2017-11-28,\"Yang, Zhongzhou; Zhang, Qiqi; Cowling, Benjamin J.; Lau, Eric H. Y.\",Sci Rep,,,,document_parses/pdf_json/9f63dea0f76ee477d2e8e5209d40179db431ab1d.json,document_parses/pmc_json/PMC5705633.xml.json,https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5705633/,\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'nTpZ93MBdq74P7QnaYOJ',\n",
       "    '_score': 12.4313545,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:58:18.250Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'unk',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': \"Does good sleep reduce early-morning off periods in patients with Parkinson's disease?\",\n",
       "     'who_covidence_id': '#32633807',\n",
       "     'sha': None,\n",
       "     'cord_uid': 'efabdz4g',\n",
       "     'url': None,\n",
       "     'source_x': 'WHO',\n",
       "     'pmc_json_files': None,\n",
       "     'pdf_json_files': None,\n",
       "     'abstract': None,\n",
       "     'journal': 'Sleep',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': None,\n",
       "     'authors': 'Suzuki, Keisuke; Fujita, Hiroaki; Okamura, Madoka; Kobayashi, Saro; Hirata, Koichi',\n",
       "     's2_id': '220385278',\n",
       "     'pubmed_id': None,\n",
       "     'publish_time': '2020',\n",
       "     'doi': None,\n",
       "     'message': 'efabdz4g,,WHO,Does good sleep reduce early-morning off periods in patients with Parkinson\\'s disease?,,,,unk,,2020,\"Suzuki, Keisuke; Fujita, Hiroaki; Okamura, Madoka; Kobayashi, Saro; Hirata, Koichi\",Sleep,,#32633807,,,,,220385278\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'wzlX93MBdq74P7QnfYTd',\n",
       "    '_score': 12.4313545,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:56:12.318Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'unk',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': \"Does good sleep reduce early-morning off periods in patients with Parkinson's disease?\",\n",
       "     'who_covidence_id': None,\n",
       "     'sha': None,\n",
       "     'cord_uid': '1vf8jd97',\n",
       "     'url': 'https://doi.org/10.1093/sleep/zsaa020; https://www.ncbi.nlm.nih.gov/pubmed/32633807/',\n",
       "     'source_x': 'Medline',\n",
       "     'pmc_json_files': None,\n",
       "     'pdf_json_files': None,\n",
       "     'abstract': None,\n",
       "     'journal': 'Sleep',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': None,\n",
       "     'authors': 'Suzuki, Keisuke; Fujita, Hiroaki; Okamura, Madoka; Kobayashi, Saro; Hirata, Koichi',\n",
       "     's2_id': '220385278',\n",
       "     'pubmed_id': '32633807',\n",
       "     'publish_time': '2020-07-06',\n",
       "     'doi': '10.1093/sleep/zsaa020',\n",
       "     'message': '1vf8jd97,,Medline,Does good sleep reduce early-morning off periods in patients with Parkinson\\'s disease?,10.1093/sleep/zsaa020,,32633807,unk,,2020-07-06,\"Suzuki, Keisuke; Fujita, Hiroaki; Okamura, Madoka; Kobayashi, Saro; Hirata, Koichi\",Sleep,,,,,,https://doi.org/10.1093/sleep/zsaa020; https://www.ncbi.nlm.nih.gov/pubmed/32633807/,220385278\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'gjlX93MBdq74P7QnAEIX',\n",
       "    '_score': 12.378336,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:55:40.196Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'no-cc',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Deactivated triple vaccine for abortus fever, Ibaraki disease and Akabane disease produced by incubation of the viruses in cell culture',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': None,\n",
       "     'cord_uid': 'bul1bptd',\n",
       "     'url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7131449/',\n",
       "     'source_x': 'PMC',\n",
       "     'pmc_json_files': None,\n",
       "     'pdf_json_files': None,\n",
       "     'abstract': None,\n",
       "     'journal': 'Vaccine',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC7131449',\n",
       "     'authors': None,\n",
       "     's2_id': None,\n",
       "     'pubmed_id': None,\n",
       "     'publish_time': '2002-11-13',\n",
       "     'doi': '10.1016/0264-410x(88)90167-3',\n",
       "     'message': 'bul1bptd,,PMC,\"Deactivated triple vaccine for abortus fever, Ibaraki disease and Akabane disease produced by incubation of the viruses in cell culture\",10.1016/0264-410x(88)90167-3,PMC7131449,,no-cc,,2002-11-13,,Vaccine,,,,,,https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7131449/,\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'Ljta93MBdq74P7QnASC1',\n",
       "    '_score': 12.140906,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:58:56.160Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'unk',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Incubation period of the coronavirus disease 2019 (COVID-19) in Busan, South Korea',\n",
       "     'who_covidence_id': '#612921',\n",
       "     'sha': None,\n",
       "     'cord_uid': 'ghdywizi',\n",
       "     'url': None,\n",
       "     'source_x': 'WHO',\n",
       "     'pmc_json_files': None,\n",
       "     'pdf_json_files': None,\n",
       "     'abstract': 'The assessment of the incubation period, which is the period between the infection and the illness onset, is essential to identify the sufficient isolation period for infectious diseases. In South Korea, a few cases of the coronavirus disease 2019 (COVID-19) were identified after the 14-day self-quarantine program, and the length of this quarantine has raised controversial issues for the Korean public health professionals. We estimated the COVID-19 incubation period using the log-normal distribution from publicly available data. The data were obtained from the press release of the Busan city department of public health and news reports. We collected and analysed information for 47 patients with a median age of 30. We estimated that the median incubation period was three days (95% Confidence Interval, 0.6-8.2). We also did not find any significant difference in the incubation period between males and females. Our findings indicate that a 14-day self-quarantine program should be sufficient to prevent spreading in the infection of suspected individuals with COVID-19 in the community.',\n",
       "     'journal': 'J. infect. chemother',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': None,\n",
       "     'authors': 'Lee, Hansol; Kim, Kyungtae; Choi, Kwonkyu; Hong, Sangbum; Son, Hyunjin; Ryu, Sukhyun',\n",
       "     's2_id': '219981022',\n",
       "     'pubmed_id': None,\n",
       "     'publish_time': '2020',\n",
       "     'doi': None,\n",
       "     'message': 'ghdywizi,,WHO,\"Incubation period of the coronavirus disease 2019 (COVID-19) in Busan, South Korea\",,,,unk,\"The assessment of the incubation period, which is the period between the infection and the illness onset, is essential to identify the sufficient isolation period for infectious diseases. In South Korea, a few cases of the coronavirus disease 2019 (COVID-19) were identified after the 14-day self-quarantine program, and the length of this quarantine has raised controversial issues for the Korean public health professionals. We estimated the COVID-19 incubation period using the log-normal distribution from publicly available data. The data were obtained from the press release of the Busan city department of public health and news reports. We collected and analysed information for 47 patients with a median age of 30. We estimated that the median incubation period was three days (95% Confidence Interval, 0.6-8.2). We also did not find any significant difference in the incubation period between males and females. Our findings indicate that a 14-day self-quarantine program should be sufficient to prevent spreading in the infection of suspected individuals with COVID-19 in the community.\",2020,\"Lee, Hansol; Kim, Kyungtae; Choi, Kwonkyu; Hong, Sangbum; Son, Hyunjin; Ryu, Sukhyun\",J. infect. chemother,,#612921,,,,,219981022\\r'}},\n",
       "   {'_index': 'metafile',\n",
       "    '_type': '_doc',\n",
       "    '_id': 'mzta93MBdq74P7Qne4jf',\n",
       "    '_score': 12.140906,\n",
       "    '_source': {'@timestamp': '2020-08-16T12:59:28.639Z',\n",
       "     'path': '/home/phani/Downloads/Data/551982_1387024_bundle_archive/metadata.csv',\n",
       "     'license': 'els-covid',\n",
       "     'host': 'phani-HP-Pavilion-x360-Convertible-14m-ba0xx',\n",
       "     'title': 'Incubation period of the coronavirus disease 2019 (COVID-19) in Busan, South Korea',\n",
       "     'who_covidence_id': None,\n",
       "     'sha': '7129c36f923b2da73c23cabcb997f83e5da33f56',\n",
       "     'cord_uid': 'uo4k3grb',\n",
       "     'url': 'https://www.sciencedirect.com/science/article/pii/S1341321X20302099?v=s5; https://api.elsevier.com/content/article/pii/S1341321X20302099; https://doi.org/10.1016/j.jiac.2020.06.018; https://www.ncbi.nlm.nih.gov/pubmed/32631735/',\n",
       "     'source_x': 'Elsevier; Medline; PMC',\n",
       "     'pmc_json_files': None,\n",
       "     'pdf_json_files': 'document_parses/pdf_json/7129c36f923b2da73c23cabcb997f83e5da33f56.json',\n",
       "     'abstract': 'Abstract The assessment of the incubation period, which is the period between the infection and the illness onset, is essential to identify the sufficient isolation period for infectious diseases. In South Korea, a few cases of the coronavirus disease 2019 (COVID-19) were identified after the 14-day self-quarantine program, and the length of this quarantine has raised controversial issues for the Korean public health professionals. We estimated the COVID-19 incubation period using the log-normal distribution from publicly available data. The data were obtained from the press release of the Busan city department of public health and news reports. We collected and analysed information for 47 patients with a median age of 30. We estimated that the median incubation period was three days (95% Confidence Interval, 0.6–8.2). We also did not find any significant difference in the incubation period between males and females. Our findings indicate that a 14-day self-quarantine program should be sufficient to prevent spreading in the infection of suspected individuals with COVID-19 in the community.',\n",
       "     'journal': 'J Infect Chemother',\n",
       "     '@version': '1',\n",
       "     'mag_id': None,\n",
       "     'arxiv_id': None,\n",
       "     'pmcid': 'PMC7311919',\n",
       "     'authors': 'Lee, Hansol; Kim, Kyungtae; Choi, Kwonkyu; Hong, Sangbum; Son, Hyunjin; Ryu, Sukhyun',\n",
       "     's2_id': '219981022',\n",
       "     'pubmed_id': '32631735',\n",
       "     'publish_time': '2020-06-24',\n",
       "     'doi': '10.1016/j.jiac.2020.06.018',\n",
       "     'message': 'uo4k3grb,7129c36f923b2da73c23cabcb997f83e5da33f56,Elsevier; Medline; PMC,\"Incubation period of the coronavirus disease 2019 (COVID-19) in Busan, South Korea\",10.1016/j.jiac.2020.06.018,PMC7311919,32631735,els-covid,\"Abstract The assessment of the incubation period, which is the period between the infection and the illness onset, is essential to identify the sufficient isolation period for infectious diseases. In South Korea, a few cases of the coronavirus disease 2019 (COVID-19) were identified after the 14-day self-quarantine program, and the length of this quarantine has raised controversial issues for the Korean public health professionals. We estimated the COVID-19 incubation period using the log-normal distribution from publicly available data. The data were obtained from the press release of the Busan city department of public health and news reports. We collected and analysed information for 47 patients with a median age of 30. We estimated that the median incubation period was three days (95% Confidence Interval, 0.6–8.2). We also did not find any significant difference in the incubation period between males and females. Our findings indicate that a 14-day self-quarantine program should be sufficient to prevent spreading in the infection of suspected individuals with COVID-19 in the community.\",2020-06-24,\"Lee, Hansol; Kim, Kyungtae; Choi, Kwonkyu; Hong, Sangbum; Son, Hyunjin; Ryu, Sukhyun\",J Infect Chemother,,,,document_parses/pdf_json/7129c36f923b2da73c23cabcb997f83e5da33f56.json,,https://www.sciencedirect.com/science/article/pii/S1341321X20302099?v=s5; https://api.elsevier.com/content/article/pii/S1341321X20302099; https://doi.org/10.1016/j.jiac.2020.06.018; https://www.ncbi.nlm.nih.gov/pubmed/32631735/,219981022\\r'}}]}}"
      ]
     },
     "execution_count": 41,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "incubation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'count': 145653,\n",
       " '_shards': {'total': 1, 'successful': 1, 'skipped': 0, 'failed': 0}}"
      ]
     },
     "execution_count": 46,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "es.count(index = \"metafile\", body= {\"query\": { \"match\": {\"title\": \"incubation periods for the disease in humans\"}}})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'count': 81400,\n",
       " '_shards': {'total': 1, 'successful': 1, 'skipped': 0, 'failed': 0}}"
      ]
     },
     "execution_count": 47,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "es.count(index = \"metafile\", body= {\"query\": { \"match\": \n",
    "                                              {\"title\": \"transmission, incubation, and environmental stability\"}}})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'count': 50181,\n",
       " '_shards': {'total': 1, 'successful': 1, 'skipped': 0, 'failed': 0}}"
      ]
     },
     "execution_count": 61,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#using match operator:\n",
    "res= es.count(index='metafile',body={'query':{'match':{'publish_time':'2020'}}})\n",
    "res"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 66,
   "metadata": {},
   "outputs": [],
   "source": [
    "#es.PUT(index = 'metafile', body={'query': {'mapping': {'properties': {'authors': {'null_value': 'no data available'}}}}})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
